From 90489f28e9b6ded2a8b4f6fe45b4ef01f63e2b31 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 5 Feb 2019 09:42:52 +0100
Subject: [PATCH] Squash of
 github/lineage-16.0(8bbbe85c)..lineage-16.0-android-9.0.0_r31(419b8472) (DO
 NOT SUBMIT)

Bill Yi (5):
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE

Luca Stefani (1):
 - Merge tag 'android-9.0.0_r31' into lineage-16.0-android-9.0.0_r31

android-build-team Robot (4):
 - Snap for 5044688 from 547d0445251a9624db1f21c165e436919dac0087 to
   pi-qpr2-release
 - Snap for 5073521 from 3ac1778e2bab8b1ca0ed22befa1a84876f58f98e to
   pi-qpr2-release
 - Snap for 5115288 from aff53256d2b3efe983d0f7ebbff83eff10eec761 to
   pi-qpr2-release
 - Snap for 5130385 from dd2146d79853afb75e1b5f7b5865dac7245adf67 to
   pi-qpr2-release

Change-Id: I7e7229eea8d4417b477423f97100a26fde85e38a

Change-Id: If0149bc0a5a31f6080259ff6debde85fbc17f661
---
 Settings/res/values-ar/strings.xml     | 24 +++++------
 Settings/res/values-as/strings.xml     |  4 +-
 Settings/res/values-ca/strings.xml     |  2 +-
 Settings/res/values-cs/strings.xml     |  2 +-
 Settings/res/values-da/strings.xml     |  4 +-
 Settings/res/values-de/strings.xml     |  6 +--
 Settings/res/values-es-rUS/strings.xml |  2 +-
 Settings/res/values-es/strings.xml     |  2 +-
 Settings/res/values-eu/strings.xml     |  4 +-
 Settings/res/values-fa/strings.xml     |  2 +-
 Settings/res/values-gl/strings.xml     |  8 ++--
 Settings/res/values-hr/strings.xml     |  4 +-
 Settings/res/values-in/strings.xml     |  2 +-
 Settings/res/values-it/strings.xml     |  8 ++--
 Settings/res/values-lo/strings.xml     |  2 +-
 Settings/res/values-mr/strings.xml     | 56 +++++++++++++-------------
 Settings/res/values-nb/strings.xml     |  2 +-
 Settings/res/values-ne/strings.xml     |  2 +-
 Settings/res/values-pl/strings.xml     |  2 +-
 Settings/res/values-pt-rBR/strings.xml |  2 +-
 Settings/res/values-pt-rPT/strings.xml |  8 ++--
 Settings/res/values-pt/strings.xml     |  2 +-
 Settings/res/values-sv/strings.xml     |  4 +-
 Settings/res/values-sw/strings.xml     |  4 +-
 Settings/res/values-ta/strings.xml     | 18 ++++-----
 Settings/res/values-th/strings.xml     |  2 +-
 Settings/res/values-tr/strings.xml     |  2 +-
 Settings/res/values-uz/strings.xml     |  4 +-
 28 files changed, 92 insertions(+), 92 deletions(-)

diff --git a/Settings/res/values-ar/strings.xml b/Settings/res/values-ar/strings.xml
index a19436e7..4df95bbb 100644
--- a/Settings/res/values-ar/strings.xml
+++ b/Settings/res/values-ar/strings.xml
@@ -316,12 +316,12 @@
     <string name="storage_format_as_public" msgid="8568317076021814054">"محو البيانات والتهيئة كوحدة تخزين قابلة للنقل"</string>
     <string name="storage_format_for_private" msgid="7095270113028182032">"التنسيق كوحدة تخزين على الجهاز"</string>
     <string name="storage_not_connected" msgid="6378589497819596096">"غير متصل"</string>
-    <string name="storage_migrate" msgid="1304058078948715456">"ترحيل البيانات إلى سعة التخزين هذه"</string>
-    <string name="storage_migrate_away" msgid="3410590501311323877">"ترحيل البيانات إلى سعة تخزين مختلفة"</string>
+    <string name="storage_migrate" msgid="1304058078948715456">"ترحيل البيانات إلى مساحة التخزين هذه"</string>
+    <string name="storage_migrate_away" msgid="3410590501311323877">"ترحيل البيانات إلى مساحة تخزين مختلفة"</string>
     <string name="storage_no_apps" msgid="579118619398668670">"ليس هناك تطبيقات للاحتفاظ بنسخ احتياطية منها."</string>
-    <string name="storage_forget" msgid="8731986045697320463">"تجاهل سعة تخزين الجهاز هذه"</string>
+    <string name="storage_forget" msgid="8731986045697320463">"تجاهل مساحة تخزين الجهاز هذه"</string>
     <string name="storage_forget_wall_of_text" msgid="8953451024930539678">"لاستخدام التطبيقات أو البيانات التي يتضمنها محرك الأقراص هذا، عليك إعادة توصيله بالجهاز. ويمكنك بدلاً من ذلك حذف وحدة التخزين هذه إذا لم يكن محرك الأقراص متاحًا.\n\nإذا اخترت \"حذف\"، فسيتم فقد كل البيانات التي يتضمنها محرك الأقراص للأبد.\n\nيمكنك إعادة تثبيت التطبيقات لاحقًا، ولكن سيتم فقد البيانات المخزنة على محرك الأقراص هذا."</string>
-    <string name="storage_device_storage_section" msgid="7797805634316822741">"سعة التخزين للجهاز"</string>
+    <string name="storage_device_storage_section" msgid="7797805634316822741">"مساحة التخزين للجهاز"</string>
     <string name="storage_removable_storage_section" msgid="3264955106565590076">"وحدة التخزين القابلة للنقل"</string>
     <string name="storage_reset_section" msgid="7324235036547192861">"إعادة الضبط"</string>
     <string name="storage_mount_success" msgid="4624254886950093443">"تم تثبيت <xliff:g id="NAME">%1$s</xliff:g>"</string>
@@ -335,10 +335,10 @@
     <string name="storage_wizard_format_as_private_title" msgid="1887035626526247913">"التهيئة كسعة تخزين على الجهاز"</string>
     <string name="storage_wizard_format_as_private_description" msgid="5814513630861655168">"يتطلب هذا تهيئة محرك أقراص USB حتى يصبح آمنًا. وبعد إجراء التهيئة بشكل آمن، لن يعمل محرك الأقراص هذا إلا مع هذا الجهاز فقط. وتؤدي التهيئة إلى محو كل البيانات المخزنة حاليًا على محرك الأقراص. ولتجنب فقد البيانات، جرّب الاحتفاظ بنسخة احتياطية منها."</string>
     <string name="storage_wizard_format_as_public_title" msgid="977475385015522171">"المحو والتهيئة"</string>
-    <string name="storage_wizard_format_as_public_description" msgid="184405877453834449">"بعد التهيئة، يمكنك استخدام محرك أقراص USB هذا على أجهزة أخرى. سيتم محو جميع البيانات. جرِّب الاحتفاظ بنسخة احتياطية أولاً من خلال نقل التطبيقات إلى سعة تخزين أخرى على الجهاز."</string>
+    <string name="storage_wizard_format_as_public_description" msgid="184405877453834449">"بعد التهيئة، يمكنك استخدام محرك أقراص USB هذا على أجهزة أخرى. سيتم محو جميع البيانات. جرِّب الاحتفاظ بنسخة احتياطية أولاً من خلال نقل التطبيقات إلى مساحة تخزين أخرى على الجهاز."</string>
     <string name="storage_wizard_format_progress_title" msgid="143911640282435570">"جارٍ تهيئة محرك أقراص USB…"</string>
     <string name="storage_wizard_format_progress_description" msgid="5320980721352781920">"قد يستغرق هذا دقيقة. يُرجى عدم إزالة محرك الأقراص."</string>
-    <string name="storage_wizard_migrate_choose_title" msgid="4247584708563925746">"اختيار سعة تخزين لترحيل البيانات إليها"</string>
+    <string name="storage_wizard_migrate_choose_title" msgid="4247584708563925746">"اختيار مساحة تخزين لترحيل البيانات إليها"</string>
     <string name="storage_wizard_migrate_confirm_title" msgid="369820166257265495">"نقل البيانات إلى <xliff:g id="NAME">%1$s</xliff:g>"</string>
     <string name="storage_wizard_migrate_confirm_description" msgid="2916497196106413598">"يمكنك نقل الصور والملفات وبيانات التطبيقات إلى <xliff:g id="NAME">%1$s</xliff:g>، قد يستغرق هذا بضع دقائق علمًا بأن بعض التطبيقات لن تعمل بشكل صحيح أثناء النقل."</string>
     <string name="storage_wizard_migrate_confirm_action_move_now" msgid="512504294066473542">"النقل الآن"</string>
@@ -354,18 +354,18 @@
     <string name="storage_wizard_back_up_apps_title" msgid="452805102278225533">"التطبيقات المخزَّنة في <xliff:g id="NAME">%1$s</xliff:g>"</string>
     <string name="storage_wizard_back_up_apps_and_data_title" msgid="1738767666148200795">"التطبيقات والبيانات المخزَّنة في <xliff:g id="NAME">%1$s</xliff:g>"</string>
     <string name="storage_wizard_back_up_apps_space_available" msgid="7209210725057470544">"<xliff:g id="SIZE">%1$s</xliff:g> متاحة"</string>
-    <string name="storage_wizard_eject_private_title" msgid="2844023177159233031">"إخراج سعة تخزين الجهاز"</string>
-    <string name="storage_wizard_eject_private_description" msgid="8966427719795472538">"سيتوقف عمل التطبيقات على سعة تخزين هذا الجهاز عند إخراجها. تمت تهيئة محرك أقراص USB هذا بحيث يعمل على هذا الجهاز فقط، ولن يعمل على أي جهاز آخر."</string>
+    <string name="storage_wizard_eject_private_title" msgid="2844023177159233031">"إخراج مساحة تخزين الجهاز"</string>
+    <string name="storage_wizard_eject_private_description" msgid="8966427719795472538">"سيتوقف عمل التطبيقات على مساحة تخزين هذا الجهاز عند إخراجها. تمت تهيئة محرك أقراص USB هذا بحيث يعمل على هذا الجهاز فقط، ولن يعمل على أي جهاز آخر."</string>
     <string name="storage_wizard_eject_progress_title" msgid="8823133087157003820">"جارٍ إخراج <xliff:g id="NAME">%1$s</xliff:g>…"</string>
-    <string name="storage_wizard_move_app_title" msgid="37057801779891287">"سعة التخزين المستخدمة"</string>
+    <string name="storage_wizard_move_app_title" msgid="37057801779891287">"مساحة التخزين المستخدمة"</string>
     <string name="storage_wizard_move_app_progress_title" msgid="7807062446517628864">"جارٍ نقل <xliff:g id="NAME">%1$s</xliff:g>…"</string>
     <string name="storage_wizard_move_app_progress_description" msgid="3693104566187904724">"يُرجى عدم إزالة محرك الأقراص أثناء النقل.\nلن يكون تطبيق <xliff:g id="APPNAME">%1$s</xliff:g> على هذا الجهاز متاحًا حتى اكتمال النقل."</string>
-    <string name="storage_wizard_forget_confirm_title" msgid="4321200084208420221">"هل تريد تجاهل سعة تخزين الجهاز؟"</string>
+    <string name="storage_wizard_forget_confirm_title" msgid="4321200084208420221">"هل تريد تجاهل مساحة تخزين الجهاز؟"</string>
     <string name="storage_wizard_forget_confirm_description" msgid="4264263982217111372">"سيتم فقد كل بياناتك المخزنة على محرك الأقراص هذا للأبد عند استخدام خيار \"حذف\". هل تريد المتابعة؟"</string>
     <string name="storage_wizard_forget_action" msgid="2532812620271112355">"حذف"</string>
     <string name="storage_new_title" msgid="3098495209500607438">"تم توصيل محرك أقراص USB"</string>
     <string name="storage_new_action_browse" msgid="9041125660441839453">"تصفُّح"</string>
-    <string name="storage_new_action_adopt" msgid="880224317112988974">"إعداد سعة تخزين الجهاز"</string>
+    <string name="storage_new_action_adopt" msgid="880224317112988974">"إعداد مساحة تخزين الجهاز"</string>
     <string name="storage_new_action_format_public" msgid="5604328981214071670">"الإعداد كوحدة تخزين قابلة للإزالة"</string>
     <string name="storage_new_action_eject" msgid="4069294152676217949">"إخراج"</string>
     <string name="storage_missing_title" msgid="2803464623922381858">"تمت إزالة <xliff:g id="NAME">%1$s</xliff:g>"</string>
@@ -683,7 +683,7 @@
     <string name="device_apps_app_management_disable_desc" msgid="1295507373252301108">"هل تريد إيقاف هذا التطبيق؟"</string>
     <string name="device_apps_app_management_enable" msgid="3246823119203444608">"تفعيل"</string>
     <string name="device_apps_app_management_enable_desc" msgid="7490219375247065314">"هل تريد تفعيل هذا التطبيق؟"</string>
-    <string name="device_apps_app_management_storage_used" msgid="4611351576179140634">"سعة التخزين المستخدمة"</string>
+    <string name="device_apps_app_management_storage_used" msgid="4611351576179140634">"مساحة التخزين المستخدمة"</string>
     <string name="device_apps_app_management_storage_used_desc" msgid="1529188874944834661">"تم استخدام <xliff:g id="SIZE">%1$s</xliff:g> في <xliff:g id="VOLUME">%2$s</xliff:g>"</string>
     <string name="device_apps_app_management_clear_data" msgid="4439114679323274127">"محو البيانات"</string>
     <string name="device_apps_app_management_clear_data_desc" msgid="4832124568689052441">"سيتم حذف جميع بيانات هذا التطبيق نهائيًا.\nويتضمن ذلك جميع الملفات والإعدادات والحسابات وقواعد البيانات، وما إلى ذلك."</string>
diff --git a/Settings/res/values-as/strings.xml b/Settings/res/values-as/strings.xml
index 0c7620f5..7b13fa2c 100644
--- a/Settings/res/values-as/strings.xml
+++ b/Settings/res/values-as/strings.xml
@@ -143,7 +143,7 @@
     <string name="system_inputs" msgid="2413907183796651102">"ইনপুটসমূহ"</string>
     <string name="system_inputs_devices" msgid="8700905419497948327">"ইনপুট আৰু ডিভাইচবোৰ"</string>
     <string name="system_home_theater_control" msgid="2053198603790384368">"হ\'ম থিয়েটাৰ নিয়ন্ত্ৰণ"</string>
-    <string name="system_accessibility" msgid="6215431399759498005">"সাধ্য় সুবিধাসমূহ"</string>
+    <string name="system_accessibility" msgid="6215431399759498005">"সাধ্য সুবিধাসমূহ"</string>
     <string name="system_developer_options" msgid="3237985523474863638">"বিকাশকৰ্তাৰ বিকল্পসমূহ"</string>
     <string name="accessibility_none" msgid="3441648940134070004">"এটাও নহয়"</string>
     <string name="system_diagnostic" msgid="7269638365915225701">"ব্যৱহাৰ আৰু চিনাক্তকৰণ"</string>
@@ -407,7 +407,7 @@
     <string name="system_services" msgid="7230571820151215779">"সেৱাসমূহ"</string>
     <string name="accessibility_service_settings" msgid="322888335240595695">"সেৱা সম্পৰ্কীয় ছেটিংসমূহ"</string>
     <string name="accessibility_toggle_high_text_contrast_preference_title" msgid="4970438261747914408">"ৰঙৰ উচ্চ প্ৰভেদযুক্ত পাঠ"</string>
-    <string name="accessibility_shortcut" msgid="6726361991387558828">"সাধ্য় সুবিধাৰ বাবে শ্বৰ্টকাটসমূহ"</string>
+    <string name="accessibility_shortcut" msgid="6726361991387558828">"সাধ্য সুবিধাসমূহৰ শ্বৰ্টকাট"</string>
     <string name="accessibility_shortcut_enable" msgid="1801656084705223078">"সাধ্য সুবিধাৰ শ্বৰ্টকাট সক্ষম কৰক"</string>
     <string name="accessibility_shortcut_service" msgid="1978686501861996601">"শ্বৰ্টকাট সেৱা"</string>
     <string name="accessibility_shortcut_description" msgid="514995490967241715">"শ্বৰ্টকাট অন হৈ থকাৰ সময়ত, আপুনি সাধ্য় সুবিধাসমূহ আৰম্ভ কৰিবলৈ বেক আৰু ডাউন দুয়োটা বুটাম ৩ ছেকেণ্ডৰ বাবে হেঁচি ৰাখিব পাৰে৷"</string>
diff --git a/Settings/res/values-ca/strings.xml b/Settings/res/values-ca/strings.xml
index 9d2eae8e..dfe3bdb2 100644
--- a/Settings/res/values-ca/strings.xml
+++ b/Settings/res/values-ca/strings.xml
@@ -477,7 +477,7 @@
     <string name="system_always_check" msgid="1866566828098009853">"Comprova sempre"</string>
     <string name="system_bt_hci_log" msgid="2425757846046504431">"Registre HCI de Bluetooth"</string>
     <string name="system_email_address" msgid="8669468986250302378">"Adreça electrònica"</string>
-    <string name="system_usb_debugging" msgid="3777895294421924323">"Depuració d\'USB"</string>
+    <string name="system_usb_debugging" msgid="3777895294421924323">"Depuració per USB"</string>
     <string name="system_allow_mock_locations" msgid="3635672039775772910">"Permet les ubicacions fictícies"</string>
     <string name="system_select_debug_app" msgid="8592021815119618485">"Selecciona l\'aplicació per depurar"</string>
     <string name="system_wait_for_debugger" msgid="4007354935701089555">"Espera el depurador"</string>
diff --git a/Settings/res/values-cs/strings.xml b/Settings/res/values-cs/strings.xml
index 5357b307..cf0c7277 100644
--- a/Settings/res/values-cs/strings.xml
+++ b/Settings/res/values-cs/strings.xml
@@ -483,7 +483,7 @@
     <string name="system_always_check" msgid="1866566828098009853">"Vždy kontrolovat"</string>
     <string name="system_bt_hci_log" msgid="2425757846046504431">"Protokolování Bluetooth HCI"</string>
     <string name="system_email_address" msgid="8669468986250302378">"E-mailová adresa"</string>
-    <string name="system_usb_debugging" msgid="3777895294421924323">"Ladění USB"</string>
+    <string name="system_usb_debugging" msgid="3777895294421924323">"Ladění přes USB"</string>
     <string name="system_allow_mock_locations" msgid="3635672039775772910">"Povolit simulované polohy"</string>
     <string name="system_select_debug_app" msgid="8592021815119618485">"Vybrat aplikaci pro ladění"</string>
     <string name="system_wait_for_debugger" msgid="4007354935701089555">"Počkat na ladicí program"</string>
diff --git a/Settings/res/values-da/strings.xml b/Settings/res/values-da/strings.xml
index 1e8deb59..3c866cf4 100644
--- a/Settings/res/values-da/strings.xml
+++ b/Settings/res/values-da/strings.xml
@@ -504,7 +504,7 @@
     <string name="system_desc_stay_awake" msgid="229588006508356238">"Deaktiver dvale"</string>
     <string name="system_desc_hdcp_checking" msgid="6879753401058891183">"Brug kun til DRM-indhold"</string>
     <string name="system_desc_hdmi_optimization" msgid="4400727262768931480">"Optimer skærmen for maksimal opløsning eller maksimal rammehastighed. Dette påvirker kun Ultra HD-skærme. En eventuel ændring af denne indstilling vil medføre en genstart af din enhed."</string>
-    <string name="system_desc_bt_hci_log" msgid="4219265203570411773">"Aktivér Bluetooth HCI snoop-logføring"</string>
+    <string name="system_desc_bt_hci_log" msgid="4219265203570411773">"Aktivér Bluetooth HCI spionlog"</string>
     <string name="system_desc_usb_debugging" msgid="5168830368191788731">"Fejlretningstilstand, når USB er tilsluttet"</string>
     <string name="system_desc_wait_for_debugger" msgid="2342968959255039368">"Fejlrettet app venter på tilknytning af fejlfinder før udførelse"</string>
     <string name="system_desc_show_layout_bounds" msgid="3449386884226708985">"Vis grænser for klip, margener osv."</string>
@@ -708,7 +708,7 @@
     <string name="device_apps_permissions" msgid="2966387603529725982">"Apptilladelser"</string>
     <string name="app_permissions_group_summary" msgid="7906834786705287901">"<xliff:g id="COUNT_0">%1$d</xliff:g> ud af <xliff:g id="COUNT_1">%2$d</xliff:g> apps har tilladelse"</string>
     <string name="bluetooth_permission_request" msgid="6649813878683015301">"Anmodning om Bluetooth-tilladelse"</string>
-    <string name="security_patch" msgid="2131617390189164481">"Niveau for sikkerhedsrettelse på Android"</string>
+    <string name="security_patch" msgid="2131617390189164481">"Seneste sikkerhedsopdatering i Android"</string>
     <string name="choose_application" msgid="5662960665577780353">"Vælg app"</string>
     <string name="experimental_preference" msgid="1137673985823581797">"(Eksperimentel)"</string>
     <string name="reboot_safemode_action" msgid="8129550438717110406">"Genstart i sikker tilstand"</string>
diff --git a/Settings/res/values-de/strings.xml b/Settings/res/values-de/strings.xml
index 4d10f298..3aa0c6fb 100644
--- a/Settings/res/values-de/strings.xml
+++ b/Settings/res/values-de/strings.xml
@@ -639,8 +639,8 @@
     <string name="pin_enter_again" msgid="1510456697040826496">"Neue PIN erneut eingeben"</string>
     <string name="pin_enter_old_pin" msgid="851029058127630869">"Alte PIN eingeben"</string>
     <string name="pin_enter_wrong_seconds" msgid="7541208202846169755">"Du hast fünfmal die falsche PIN eingegeben.\nVersuche es in <xliff:g id="RELATIVE_TIME_SPAN">%1$d</xliff:g> Sekunden erneut."</string>
-    <string name="pin_toast_wrong" msgid="4628358195224633248">"Falsche PIN. Bitte versuche es erneut."</string>
-    <string name="pin_toast_not_match" msgid="5886041789737632996">"Die PIN stimmt nicht. Bitte versuche es erneut."</string>
+    <string name="pin_toast_wrong" msgid="4628358195224633248">"Falsche PIN. Bitte versuche es noch einmal."</string>
+    <string name="pin_toast_not_match" msgid="5886041789737632996">"Die PIN stimmt nicht. Bitte versuche es noch einmal."</string>
     <string name="wifi_setup_input_password" msgid="1596168874369038536">"Passwort für <xliff:g id="SSID">%1$s</xliff:g> eingeben"</string>
     <string name="wifi_setup_connection_success" msgid="3846993789166071184">"Die Verbindung wurde hergestellt"</string>
     <string name="wifi_setup_save_success" msgid="4785799603638549254">"Die Konfiguration wurde gespeichert"</string>
@@ -719,7 +719,7 @@
     <string name="app_permission_summary_allowed" msgid="1618833359547333078">"Zugelassen"</string>
     <string name="app_permission_summary_not_allowed" msgid="3667663331476577913">"Nicht zugelassen"</string>
     <string name="usage_access" msgid="1914242907684452328">"Zugriff auf Nutzungsdaten"</string>
-    <string name="usage_access_description" msgid="6290001769893747651">"Der Zugriff auf die Nutzungsdaten ermöglicht es einer App, verschiedene Informationen zu erfassen. Unter anderem, welche anderen Apps du wie oft verwendest, welchen Mobilfunkanbieter du nutzt und welche Spracheinstellungen du festgelegt hast."</string>
+    <string name="usage_access_description" msgid="6290001769893747651">"Der Zugriff auf die Nutzungsdaten ermöglicht es einer App, verschiedene Informationen zu erfassen – welche anderen Apps du wie oft verwendest, welchen Mobilfunkanbieter du nutzt, welche Spracheinstellungen du festgelegt hast usw."</string>
     <string name="high_power_apps" msgid="7796096221999495077">"Optimierter Energieverbrauch"</string>
     <string name="high_power_apps_description" msgid="1990016812115792282">"Energieverbrauch von Apps optimieren"</string>
     <string name="high_power_apps_empty" msgid="1355906551507153470">"Es müssen keine Apps optimiert werden"</string>
diff --git a/Settings/res/values-es-rUS/strings.xml b/Settings/res/values-es-rUS/strings.xml
index 8c13e46e..3e0d0126 100644
--- a/Settings/res/values-es-rUS/strings.xml
+++ b/Settings/res/values-es-rUS/strings.xml
@@ -719,7 +719,7 @@
     <string name="app_permission_summary_allowed" msgid="1618833359547333078">"Permitido"</string>
     <string name="app_permission_summary_not_allowed" msgid="3667663331476577913">"No permitido"</string>
     <string name="usage_access" msgid="1914242907684452328">"Acceso a los datos de uso"</string>
-    <string name="usage_access_description" msgid="6290001769893747651">"El acceso a los datos de uso permite que una aplicación realice un seguimiento de qué otras aplicaciones usas y de la frecuencia con que las usas, además de conocer tu proveedor, configuración de idioma y otros detalles."</string>
+    <string name="usage_access_description" msgid="6290001769893747651">"El acceso a los datos de uso permite que una aplicación controle qué otras aplicaciones utilizas y con qué frecuencia, así como el operador, la configuración de idioma y otros detalles."</string>
     <string name="high_power_apps" msgid="7796096221999495077">"Optimización de energía"</string>
     <string name="high_power_apps_description" msgid="1990016812115792282">"Optimiza el uso de energía de las apps"</string>
     <string name="high_power_apps_empty" msgid="1355906551507153470">"No es necesario optimizar ninguna app"</string>
diff --git a/Settings/res/values-es/strings.xml b/Settings/res/values-es/strings.xml
index 84331f7e..63a5b3c1 100644
--- a/Settings/res/values-es/strings.xml
+++ b/Settings/res/values-es/strings.xml
@@ -176,7 +176,7 @@
     <string name="device_status" msgid="4379924151946190768">"Estado"</string>
     <string name="device_status_summary" msgid="180586140242936323">"Red, números de serie y otra información"</string>
     <string name="manual" msgid="4957388668881541669">"Manual"</string>
-    <string name="regulatory_information" msgid="5448865708097003189">"Normativas"</string>
+    <string name="regulatory_information" msgid="5448865708097003189">"Información normativa"</string>
     <string name="device_feedback" msgid="209146642122579479">"Enviar opinión sobre el dispositivo"</string>
     <string name="fcc_equipment_id" msgid="8295142996591094301">"ID de equipo"</string>
     <string name="baseband_version" msgid="2229081246529676107">"Versión de banda base"</string>
diff --git a/Settings/res/values-eu/strings.xml b/Settings/res/values-eu/strings.xml
index b9d9ad6c..c8c7c923 100644
--- a/Settings/res/values-eu/strings.xml
+++ b/Settings/res/values-eu/strings.xml
@@ -176,7 +176,7 @@
     <string name="device_status" msgid="4379924151946190768">"Egoera"</string>
     <string name="device_status_summary" msgid="180586140242936323">"Sarea, serie-zenbakiak eta bestelako informazioa"</string>
     <string name="manual" msgid="4957388668881541669">"Eskuliburua"</string>
-    <string name="regulatory_information" msgid="5448865708097003189">"Araudiak"</string>
+    <string name="regulatory_information" msgid="5448865708097003189">"Arauei buruzko informazioa"</string>
     <string name="device_feedback" msgid="209146642122579479">"Bidali gailu honi buruzko iritzia"</string>
     <string name="fcc_equipment_id" msgid="8295142996591094301">"Ekipoaren IDa"</string>
     <string name="baseband_version" msgid="2229081246529676107">"Oinarri-bandaren bertsioa"</string>
@@ -461,7 +461,7 @@
     <string name="system_speech_rate" msgid="8099856582088545576">"Hizketaren abiadura"</string>
     <string name="system_play_sample" msgid="7187293932891811148">"Erreproduzitu lagina"</string>
     <string name="system_install_voice_data" msgid="8920883886563714355">"Instalatu ahotsaren datuak"</string>
-    <string name="system_general" msgid="3535958631591158282">"Orokorra"</string>
+    <string name="system_general" msgid="3535958631591158282">"Orokorrak"</string>
     <string name="system_debugging" msgid="7204447785851191331">"Arazketa"</string>
     <string name="system_input" msgid="6164987320529039626">"Sarrera"</string>
     <string name="system_drawing" msgid="6573389872154957039">"Marrazkia"</string>
diff --git a/Settings/res/values-fa/strings.xml b/Settings/res/values-fa/strings.xml
index b5603b47..e222d20c 100644
--- a/Settings/res/values-fa/strings.xml
+++ b/Settings/res/values-fa/strings.xml
@@ -476,7 +476,7 @@
     <string name="system_check_for_drm_content_only" msgid="8716366734950874267">"فقط بررسی محتوای DRM"</string>
     <string name="system_always_check" msgid="1866566828098009853">"همیشه بررسی شود"</string>
     <string name="system_bt_hci_log" msgid="2425757846046504431">"گزارش‌گیری HCI بلوتوث"</string>
-    <string name="system_email_address" msgid="8669468986250302378">"نشانی رایانامه"</string>
+    <string name="system_email_address" msgid="8669468986250302378">"نشانی ایمیل"</string>
     <string name="system_usb_debugging" msgid="3777895294421924323">"اشکال‌زدایی USB"</string>
     <string name="system_allow_mock_locations" msgid="3635672039775772910">"مکان‌های کاذب مجاز هستند"</string>
     <string name="system_select_debug_app" msgid="8592021815119618485">"انتخاب برنامه اشکال‌زدایی"</string>
diff --git a/Settings/res/values-gl/strings.xml b/Settings/res/values-gl/strings.xml
index 52fd511b..5d28eef3 100644
--- a/Settings/res/values-gl/strings.xml
+++ b/Settings/res/values-gl/strings.xml
@@ -399,11 +399,11 @@
     <string name="google_location_services_title" msgid="3382840371861269146">"Servizos de localización de Google"</string>
     <string name="third_party_location_services_title" msgid="4089136801976399147">"Servizos de localización de terceiros"</string>
     <string name="location_reporting" msgid="3054236612250063073">"Informes de localización"</string>
-    <string name="location_history" msgid="2946680949724247804">"Historial de localizacións"</string>
+    <string name="location_history" msgid="2946680949724247804">"Historial de localización"</string>
     <string name="location_reporting_desc" msgid="6256424995139430823">"Google utiliza esta función en produtos como Google Now e Google Maps. A activación dos Informes de localización permite aos produtos de Google que utilizan esta función almacenar e usar os datos de localización máis recentes do teu dispositivo en conexión coa túa conta de Google."</string>
-    <string name="location_history_desc" msgid="3861249867748294369">"Cando o Historial de localizacións está activado para esta conta, Google pode almacenar os datos de localización do teu dispositivo para que os utilicen as túas aplicacións.\n\nPor exemplo, Google Maps pode facilitarche indicacións e Google Now informarte acerca do tráfico do teu desprazamento diario.\n\nPodes desactivar o Historial de localizacións cando queiras, pero ao facelo non se eliminará. Para ver e xestionar o teu Historial de localizacións, visita a páxina maps.google.com/locationhistory."</string>
-    <string name="delete_location_history_title" msgid="5804802702252280549">"Eliminar Historial de localizacións"</string>
-    <string name="delete_location_history_desc" msgid="4142806827237093397">"Eliminaranse todos os datos que o Historial de localizacións almacenou deste dispositivo nesta conta de Google. Esta acción non se pode desfacer. Algunhas aplicacións, incluída Google Now, deixarán de funcionar."</string>
+    <string name="location_history_desc" msgid="3861249867748294369">"Cando o historial de localización está activado para esta conta, Google pode almacenar os datos de localización do teu dispositivo para que os utilicen as túas aplicacións.\n\nPor exemplo, Google Maps pode facilitarche indicacións e Google Now informarte acerca do tráfico do teu desprazamento diario.\n\nPodes desactivar o historial de localización cando queiras, pero ao facelo non se eliminará. Para ver e xestionar o teu historial de localización, visita a páxina maps.google.com/locationhistory."</string>
+    <string name="delete_location_history_title" msgid="5804802702252280549">"Eliminar historial de localización"</string>
+    <string name="delete_location_history_desc" msgid="4142806827237093397">"Eliminaranse todos os datos que o historial de localización almacenou deste dispositivo nesta conta de Google. Esta acción non se pode desfacer. Algunhas aplicacións, incluída Google Now, deixarán de funcionar."</string>
     <string name="system_services" msgid="7230571820151215779">"Servizos"</string>
     <string name="accessibility_service_settings" msgid="322888335240595695">"Configuración dos servizos"</string>
     <string name="accessibility_toggle_high_text_contrast_preference_title" msgid="4970438261747914408">"Texto de alto contraste"</string>
diff --git a/Settings/res/values-hr/strings.xml b/Settings/res/values-hr/strings.xml
index a4ef1f4d..0d8de4cb 100644
--- a/Settings/res/values-hr/strings.xml
+++ b/Settings/res/values-hr/strings.xml
@@ -486,8 +486,8 @@
     <string name="system_wait_for_debugger" msgid="4007354935701089555">"Čekanje na program za otklanjanje pogrešaka"</string>
     <string name="system_verify_apps_over_usb" msgid="7340861918835270609">"Potvrdi aplikacije putem USB-a"</string>
     <string name="system_desc_verify_apps_over_usb" msgid="6722973410408262393">"Provjerite uzrokuju li aplikacije instalirane putem ADB-a/ADT-a poteškoće"</string>
-    <string name="system_wifi_verbose_logging" msgid="3474525355071613242">"Opširni zapisnik Wi-Fi-ja"</string>
-    <string name="system_desc_wifi_verbose_logging" msgid="1559904751873455868">"Omogući opširni zapisnik Wi-Fi-ja"</string>
+    <string name="system_wifi_verbose_logging" msgid="3474525355071613242">"Opširni zapisnik Wi-Fija"</string>
+    <string name="system_desc_wifi_verbose_logging" msgid="1559904751873455868">"Omogući opširni zapisnik Wi-Fija"</string>
     <string name="system_show_touches" msgid="2179121368437801757">"Prikazivanje dodira"</string>
     <string name="system_pointer_location" msgid="2673764026490179227">"Lokacija pokazivača"</string>
     <string name="system_show_layout_bounds" msgid="645732439882291943">"Prikazivanje granica rasporeda"</string>
diff --git a/Settings/res/values-in/strings.xml b/Settings/res/values-in/strings.xml
index 0c0c4626..dd0b1bd6 100644
--- a/Settings/res/values-in/strings.xml
+++ b/Settings/res/values-in/strings.xml
@@ -729,7 +729,7 @@
     <string name="manage_notification_access_title" msgid="2051765294087620434">"Akses notifikasi"</string>
     <string name="no_notification_listeners" msgid="4583789599075326886">"Tidak ada aplikasi terinstal yang meminta akses notifikasi."</string>
     <string name="notification_listener_security_warning_summary" msgid="3647205635922300578">"Aplikasi ini dapat membaca semua notifikasi, termasuk informasi pribadi seperti nama kontak dan teks pesan yang Anda terima. Aplikasi ini juga dapat menutup notifikasi atau memicu tombol tindakan yang dimilikinya."</string>
-    <string name="system_alert_window_settings" msgid="2069031369694448339">"Tampilkan di atas aplikasi lain"</string>
+    <string name="system_alert_window_settings" msgid="2069031369694448339">"Tampilkan di atas apl lain"</string>
     <string name="permit_draw_overlay" msgid="2400084757761408906">"Izinkan ditampilkan di atas aplikasi lain"</string>
     <string name="allow_overlay_description" msgid="1946279120428813474">"Izinkan aplikasi ditampilkan di atas aplikasi lain yang sedang digunakan. Setelan ini dapat mengganggu penggunaan aplikasi tersebut atau mengubah cara tampilannya atau perilakunya."</string>
     <string name="write_system_settings" msgid="2730923656857824907">"Ubah setelan sistem"</string>
diff --git a/Settings/res/values-it/strings.xml b/Settings/res/values-it/strings.xml
index 5021a0b3..a7fac386 100644
--- a/Settings/res/values-it/strings.xml
+++ b/Settings/res/values-it/strings.xml
@@ -400,10 +400,10 @@
     <string name="third_party_location_services_title" msgid="4089136801976399147">"Servizi di geolocalizzazione di terze parti"</string>
     <string name="location_reporting" msgid="3054236612250063073">"Segnalazione della posizione"</string>
     <string name="location_history" msgid="2946680949724247804">"Cronologia delle posizioni"</string>
-    <string name="location_reporting_desc" msgid="6256424995139430823">"Google utilizza questa funzione in prodotti come Google Now e Google Maps. Se attivi la Segnalazione della posizione, ogni prodotto Google che utilizza questa funzione può memorizzare e utilizzare i dati sulla posizione più recenti del dispositivo associati al tuo account Google."</string>
+    <string name="location_reporting_desc" msgid="6256424995139430823">"Google utilizza questa funzione in prodotti come Google Now e Google Maps. Se attivi la Segnalazione della posizione, ogni prodotto Google che utilizza questa funzione può memorizzare e utilizzare i dati sulla posizione più recenti del dispositivo associati al tuo Account Google."</string>
     <string name="location_history_desc" msgid="3861249867748294369">"Quando la Cronologia delle posizioni è attiva per l\'account, Google può memorizzare i dati sulla posizione del dispositivo che verrano utilizzati dalle tue app.\n\nAd esempio, Google Maps può darti indicazioni stradali e Google Now può comunicarti il traffico lungo il tuo tragitto giornaliero.\n\nPuoi disattivare la Cronologia delle posizioni in qualsiasi momento, ma non verrà eliminata. Per visualizzare e gestire la Cronologia delle posizioni, visita il sito maps.google.com/locationhistory."</string>
     <string name="delete_location_history_title" msgid="5804802702252280549">"Elimina cronologia delle posizioni"</string>
-    <string name="delete_location_history_desc" msgid="4142806827237093397">"Verrà eliminata l\'intera Cronologia delle posizioni memorizzata sul dispositivo per l\'account Google. Non puoi annullare l\'eliminazione. Alcune app, inclusa Google Now, non funzioneranno più."</string>
+    <string name="delete_location_history_desc" msgid="4142806827237093397">"Verrà eliminata l\'intera Cronologia delle posizioni memorizzata sul dispositivo per l\'Account Google. Non puoi annullare l\'eliminazione. Alcune app, inclusa Google Now, non funzioneranno più."</string>
     <string name="system_services" msgid="7230571820151215779">"Servizi"</string>
     <string name="accessibility_service_settings" msgid="322888335240595695">"Impostazioni servizio"</string>
     <string name="accessibility_toggle_high_text_contrast_preference_title" msgid="4970438261747914408">"Testo alto contrasto"</string>
@@ -685,7 +685,7 @@
     <string name="privacy_backup_account" msgid="9059939814827165020">"Account di backup"</string>
     <string name="privacy_automatic_restore" msgid="1409885318093199628">"Ripristino automatico"</string>
     <string name="factory_reset_device" msgid="7422459553658815011">"Reimposta dispositivo"</string>
-    <string name="factory_reset_description" msgid="5381170999141355891">"Questa operazione comporta l\'eliminazione di tutti i dati dalla memoria interna del dispositivo, tra cui: il tuo account Google, dati e impostazioni del sistema e delle app, nonché le app scaricate."</string>
+    <string name="factory_reset_description" msgid="5381170999141355891">"Questa operazione comporta l\'eliminazione di tutti i dati dalla memoria interna del dispositivo, tra cui: il tuo Account Google, dati e impostazioni del sistema e delle app, nonché le app scaricate."</string>
     <string name="confirm_factory_reset_description" msgid="1049842658625287303">"Cancellare tutte le informazioni personali e le app scaricate? Questa azione è irreversibile."</string>
     <string name="confirm_factory_reset_device" msgid="8923287482399582268">"Cancella tutto"</string>
     <string name="select_device_name_title" msgid="5165800500130943604">"Scegli un nome per il tuo dispositivo <xliff:g id="DEVICEMODEL">%1$s</xliff:g>"</string>
@@ -723,7 +723,7 @@
     <string name="high_power_apps" msgid="7796096221999495077">"Ottimizzazione energia"</string>
     <string name="high_power_apps_description" msgid="1990016812115792282">"Ottimizza il consumo di energia delle app"</string>
     <string name="high_power_apps_empty" msgid="1355906551507153470">"Nessuna app da ottimizzare"</string>
-    <string name="high_power_on" msgid="7725180078784796673">"Non ottimizzate"</string>
+    <string name="high_power_on" msgid="7725180078784796673">"Senza ottimizzazione"</string>
     <string name="high_power_off" msgid="308272752152871252">"Ottimizzazione del consumo di energia"</string>
     <string name="high_power_system" msgid="7815843517656137656">"Ottimizzazione dell\'energia non disponibile"</string>
     <string name="manage_notification_access_title" msgid="2051765294087620434">"Accesso alle notifiche"</string>
diff --git a/Settings/res/values-lo/strings.xml b/Settings/res/values-lo/strings.xml
index 1f2f3a9e..c29b8bb7 100644
--- a/Settings/res/values-lo/strings.xml
+++ b/Settings/res/values-lo/strings.xml
@@ -52,7 +52,7 @@
     <string name="account_sync" msgid="773808124331714965">"​ເລືອກ​ແອັບຯ​ທີ່​ຊິ້ງ​​​ຂໍ້​ມູນແລ້ວ"</string>
     <string name="sync_now" msgid="8732244492880016471">"ຊິ້ງຂໍ້ມູນດຽວນີ້"</string>
     <string name="sync_in_progress" msgid="1330721038775646022">"ກຳລັງຊິ້ງຂໍ້ມູນ..."</string>
-    <string name="last_synced" msgid="6178494892357077274">"ຊິ້ງຂໍ້​ມູນ​ເທື່ອ​ສຸດ​ທ້າຍ​ເມື່ອ <xliff:g id="TIME">%1s</xliff:g>"</string>
+    <string name="last_synced" msgid="6178494892357077274">"ຊິ້ງຂໍ້ມູນຫຼ້າສຸດ <xliff:g id="TIME">%1s</xliff:g>"</string>
     <string name="sync_disabled" msgid="3089360175872892781">"ປິດ​ນຳໃຊ້​ແລ້ວ"</string>
     <string name="account_remove" msgid="4313580520737295272">"ລຶບບັນຊີ"</string>
     <string name="account_remove_failed" msgid="980654375770693544">"ບໍ່​ສາ​ມາດ​ລຶບ​ບັ​ນ​ຊີ​ໄດ້"</string>
diff --git a/Settings/res/values-mr/strings.xml b/Settings/res/values-mr/strings.xml
index bc7838e9..32aaf511 100644
--- a/Settings/res/values-mr/strings.xml
+++ b/Settings/res/values-mr/strings.xml
@@ -117,7 +117,7 @@
     <string name="accessory_change_name" msgid="8958916008739792785">"नाव बदला"</string>
     <string name="accessory_change_name_title" msgid="4630819456923061184">"या उपसाधनासाठी एक नवीन नाव प्रविष्‍ट करा"</string>
     <string name="accessories_add_title" msgid="6940361421891873321">"उपसाधने शोधत आहे..."</string>
-    <string name="accessories_add_bluetooth_inst" msgid="5892952745770640383">"आपले ब्लूटूथ डिव्हाइसेस जोडण्यापूर्वी, ते जोडणी मोडमध्ये असल्याचे सुनिश्चित करा"</string>
+    <string name="accessories_add_bluetooth_inst" msgid="5892952745770640383">"तुमचे ब्लूटूथ डिव्हाइसेस जोडण्यापूर्वी, ते जोडणी मोडमध्ये असल्याचे सुनिश्चित करा"</string>
     <string name="accessories_autopair_msg" msgid="8618829246837821752">"एक डिव्हाइस आढळले आणि ते <xliff:g id="COUNTDOWN">%1$s</xliff:g> सेकंदांमध्ये स्वयंचलितपणे जोडले जाईल"</string>
     <string name="error_action_not_supported" msgid="300552308104354666">"ही क्रिया समर्थित नाही"</string>
     <string name="bluetooth_pairing_request" msgid="4523088117471287665">"ब्लूटूथ जोडणी विनंती"</string>
@@ -161,12 +161,12 @@
     <string name="about_serial" msgid="2723560490027171187">"अनुक्रमांक"</string>
     <string name="about_build" msgid="9148789767199785676">"बिल्ड"</string>
     <plurals name="show_dev_countdown" formatted="false" msgid="5361765297521871966">
-      <item quantity="one">आपण अाता विकासक होण्यापासून <xliff:g id="STEP_COUNT_1">%1$d</xliff:g> चरण दूर आहात</item>
-      <item quantity="other">आपण विकासक होण्यापासून<xliff:g id="STEP_COUNT_1">%1$d</xliff:g> चरणे दूर आहात</item>
+      <item quantity="one">तुम्ही अाता विकासक होण्यापासून <xliff:g id="STEP_COUNT_1">%1$d</xliff:g> चरण दूर आहात</item>
+      <item quantity="other">तुम्ही विकासक होण्यापासून<xliff:g id="STEP_COUNT_1">%1$d</xliff:g> चरणे दूर आहात</item>
     </plurals>
     <string name="about_ads" msgid="3364786994943748337">"जाहिराती"</string>
-    <string name="show_dev_on" msgid="8997172736024819952">"आता आपण एक विकासक आहात!"</string>
-    <string name="show_dev_already" msgid="7525967300421794483">"आवश्यकता नाही, आपण आधीपासूनच एक विकसक आहात"</string>
+    <string name="show_dev_on" msgid="8997172736024819952">"आता तुम्ही एक विकासक आहात!"</string>
+    <string name="show_dev_already" msgid="7525967300421794483">"आवश्यकता नाही, तुम्ही आधीपासूनच एक विकसक आहात"</string>
     <string name="device_info_default" msgid="4519181343309136050">"अज्ञात"</string>
     <string name="selinux_status" msgid="8000422283341040360">"SELinux स्थिती"</string>
     <string name="selinux_status_disabled" msgid="4925733135520787807">"अक्षम केले"</string>
@@ -308,7 +308,7 @@
     <string name="storage_migrate_away" msgid="3410590501311323877">"भिन्न संचयावर डेटा स्थलांतरित करा"</string>
     <string name="storage_no_apps" msgid="579118619398668670">"बॅकअप घेण्यासाठी कोणतेही अॅप्स नाहीत"</string>
     <string name="storage_forget" msgid="8731986045697320463">"हा डिव्हाइस स्टोरेज विसरा"</string>
-    <string name="storage_forget_wall_of_text" msgid="8953451024930539678">"या ड्राइव्ह मध्‍ये असलेले अ‍ॅप्स किंवा डेटा वापरण्‍यासाठी, त्यास त्यामध्‍ये पुन्हा प्लग करा. वैकल्पिकपणे, ड्राइव्ह उपलब्ध नसल्यास आपण हे संचयन विसरणे निवडू शकता.\n\nआपण विसरा निवडल्‍यास, ड्राइव्हमध्‍ये असलेला सर्व डेटा कायमचा गमावला जाईल.\n\nआपण अ‍ॅप्‍स नंतर पुन्हा इंस्टॉल करू शकता परंतु या ड्राइव्हवर स्टोअर केलेला डेटा गमावला जाईल."</string>
+    <string name="storage_forget_wall_of_text" msgid="8953451024930539678">"या ड्राइव्ह मध्‍ये असलेले अ‍ॅप्स किंवा डेटा वापरण्‍यासाठी, त्यास त्यामध्‍ये पुन्हा प्लग करा. वैकल्पिकपणे, ड्राइव्ह उपलब्ध नसल्यास तुम्ही हे संचयन विसरणे निवडू शकता.\n\nतुम्ही विसरा निवडल्‍यास, ड्राइव्हमध्‍ये असलेला सर्व डेटा कायमचा गमावला जाईल.\n\nतुम्ही अ‍ॅप्‍स नंतर पुन्हा इंस्टॉल करू शकता परंतु या ड्राइव्हवर स्टोअर केलेला डेटा गमावला जाईल."</string>
     <string name="storage_device_storage_section" msgid="7797805634316822741">"डिव्‍हाइस संचयन"</string>
     <string name="storage_removable_storage_section" msgid="3264955106565590076">"काढण्‍यायोग्य संचयन"</string>
     <string name="storage_reset_section" msgid="7324235036547192861">"रीसेट करा"</string>
@@ -323,12 +323,12 @@
     <string name="storage_wizard_format_as_private_title" msgid="1887035626526247913">"डिव्हाइस स्टोरेज म्हणून फॉरमॅट करा"</string>
     <string name="storage_wizard_format_as_private_description" msgid="5814513630861655168">"हे सुरक्षित करण्यासाठी यास USB ड्राइव्ह फॉर्मेट करणे आवश्यक आहे. सुरक्षितपणे फॉर्मेट केल्यानंतर, हा ड्राइव्ह केवळ या डिव्हाइससह कार्य करेल. फॉर्मेट करण्यामुळे ड्राइव्हवर सध्या स्टोअर केलेला सर्व डेटा मिटविला जातो. डेटा गमावणे टाळण्यासाठी, त्याचा बॅकअप घेण्याचा विचार करा."</string>
     <string name="storage_wizard_format_as_public_title" msgid="977475385015522171">"मिटवा आणि फॉर्मेट करा"</string>
-    <string name="storage_wizard_format_as_public_description" msgid="184405877453834449">"फॉरमॅट केल्यानंतर, आपण हा USB ड्राइव्ह अन्य डिव्हाइसेसमध्ये वापरू शकता. सर्व डेटा मिटविला जाईल. अन्य डिव्हाइस संचयावर अ‍ॅप्स हलवून प्रथम बॅकअप घेण्याचा विचार करा."</string>
+    <string name="storage_wizard_format_as_public_description" msgid="184405877453834449">"फॉरमॅट केल्यानंतर, तुम्ही हा USB ड्राइव्ह अन्य डिव्हाइसेसमध्ये वापरू शकता. सर्व डेटा मिटविला जाईल. अन्य डिव्हाइस संचयावर अ‍ॅप्स हलवून प्रथम बॅकअप घेण्याचा विचार करा."</string>
     <string name="storage_wizard_format_progress_title" msgid="143911640282435570">"USB ड्राइव्ह फॉर्मेट करत आहे…"</string>
     <string name="storage_wizard_format_progress_description" msgid="5320980721352781920">"यास एक क्षण लागू शकतो. कृपया ड्राइव्ह काढू नका."</string>
     <string name="storage_wizard_migrate_choose_title" msgid="4247584708563925746">"डेटा ज्यावर स्थलांतरित करायचा तो स्टोरेज निवडा"</string>
     <string name="storage_wizard_migrate_confirm_title" msgid="369820166257265495">"डेटा <xliff:g id="NAME">%1$s</xliff:g> वर हलवा"</string>
-    <string name="storage_wizard_migrate_confirm_description" msgid="2916497196106413598">"आपले फोटो, फायली आणि अ‍ॅप डेटा <xliff:g id="NAME">%1$s</xliff:g> वर हलवा. यास काही मिनिटे लागू शकतात. हलविण्‍यादरम्यान काही अ‍ॅप्स योग्यरित्या कार्य करणार नाहीत."</string>
+    <string name="storage_wizard_migrate_confirm_description" msgid="2916497196106413598">"तुमचे फोटो, फायली आणि अ‍ॅप डेटा <xliff:g id="NAME">%1$s</xliff:g> वर हलवा. यास काही मिनिटे लागू शकतात. हलविण्‍यादरम्यान काही अ‍ॅप्स योग्यरित्या कार्य करणार नाहीत."</string>
     <string name="storage_wizard_migrate_confirm_action_move_now" msgid="512504294066473542">"आता हलवा"</string>
     <string name="storage_wizard_migrate_confirm_action_move_later" msgid="7838503647349016818">"नंतर हलवा"</string>
     <string name="storage_wizard_migrate_toast_success" msgid="2634978204935628045">"डेटा <xliff:g id="NAME">%1$s</xliff:g> वर स्थलांतरित केला"</string>
@@ -336,11 +336,11 @@
     <string name="storage_wizard_migrate_progress_title" msgid="8063255124619498728">"डेटा <xliff:g id="NAME">%1$s</xliff:g> वर हलवित आहोत…"</string>
     <string name="storage_wizard_migrate_progress_description" msgid="3337889465416363170">"यास क्षणभर लागू शकतो. कृपया ड्राइव्ह काढू नका.\nहलविण्‍यादरम्यान काही अ‍ॅप्स योग्यरित्या कार्य करणार नाहीत."</string>
     <string name="storage_wizard_format_slow_title" msgid="2483634437996887661">"हा ड्राइव्ह धीमा झाल्यासारखे दिसते आहे."</string>
-    <string name="storage_wizard_format_slow_summary" msgid="1949177940762639885">"आपण सुरु ठेवू शकता, परंतु या स्थानावर हलविलेल्या अॅप्समध्ये अडथळा येऊ शकतो आणि डेटा स्थानांतरणास बराच वेळ लागू शकतो. अधिक चांगल्या कार्यप्रदर्शनासाठी आणखी जलद ड्राइव्ह वापरण्याचा विचार करा."</string>
+    <string name="storage_wizard_format_slow_summary" msgid="1949177940762639885">"तुम्ही सुरु ठेवू शकता, परंतु या स्थानावर हलविलेल्या अॅप्समध्ये अडथळा येऊ शकतो आणि डेटा स्थानांतरणास बराच वेळ लागू शकतो. अधिक चांगल्या कार्यप्रदर्शनासाठी आणखी जलद ड्राइव्ह वापरण्याचा विचार करा."</string>
     <string name="storage_wizard_format_action" msgid="3986485572528374905">"फॉर्मेट करा"</string>
     <string name="storage_wizard_backup_apps_action" msgid="7246092320517630760">"बॅकअप अ‍ॅप्‍स"</string>
-    <string name="storage_wizard_back_up_apps_title" msgid="452805102278225533">"<xliff:g id="NAME">%1$s</xliff:g> मध्ये स्टोअर केलेले अॅप्स"</string>
-    <string name="storage_wizard_back_up_apps_and_data_title" msgid="1738767666148200795">"<xliff:g id="NAME">%1$s</xliff:g> मध्ये स्टोअर केलेले अॅप्स आणि डेटा"</string>
+    <string name="storage_wizard_back_up_apps_title" msgid="452805102278225533">"<xliff:g id="NAME">%1$s</xliff:g> मध्ये स्टोअर केलेली अॅप्स"</string>
+    <string name="storage_wizard_back_up_apps_and_data_title" msgid="1738767666148200795">"<xliff:g id="NAME">%1$s</xliff:g> मध्ये स्टोअर केलेली अॅप्स आणि डेटा"</string>
     <string name="storage_wizard_back_up_apps_space_available" msgid="7209210725057470544">"<xliff:g id="SIZE">%1$s</xliff:g> उपलब्ध"</string>
     <string name="storage_wizard_eject_private_title" msgid="2844023177159233031">"डिव्हाइस स्टोरेज बाहेर काढा"</string>
     <string name="storage_wizard_eject_private_description" msgid="8966427719795472538">"हा डिव्हाइस स्टोरेज बाहेर काढल्यानंतर त्यावरील अ‍ॅप्स कार्य करणे थांबवतील. हा USB ड्राइव्ह केवळ या डिव्हाइसवर कार्य करण्यासाठी फॉरमॅट केला आहे. तो अन्य कोणत्याही डिव्हाइसवर कार्य करणार नाही."</string>
@@ -349,7 +349,7 @@
     <string name="storage_wizard_move_app_progress_title" msgid="7807062446517628864">"<xliff:g id="NAME">%1$s</xliff:g> हलवित आहे…"</string>
     <string name="storage_wizard_move_app_progress_description" msgid="3693104566187904724">"हलविण्यादरम्यान ड्राइव्ह काढू नका.\nहलविणे पूर्ण होईपर्यंत या डिव्हाइसवरील <xliff:g id="APPNAME">%1$s</xliff:g> अॅप उपलब्ध नसेल."</string>
     <string name="storage_wizard_forget_confirm_title" msgid="4321200084208420221">"डिव्हाइस स्टोरेज विसरला?"</string>
-    <string name="storage_wizard_forget_confirm_description" msgid="4264263982217111372">"या ड्राइव्हमध्‍ये स्टोअर केलेला आपला सर्व डेटा \'विसरा\' सह कायमचा गमावला जाईल. आपण पुढे सुरु ठेवू इच्छिता?"</string>
+    <string name="storage_wizard_forget_confirm_description" msgid="4264263982217111372">"या ड्राइव्हमध्‍ये स्टोअर केलेला तुमचा सर्व डेटा \'विसरा\' सह कायमचा गमावला जाईल. तुम्ही पुढे सुरु ठेवू इच्छिता?"</string>
     <string name="storage_wizard_forget_action" msgid="2532812620271112355">"विसरा"</string>
     <string name="storage_new_title" msgid="3098495209500607438">"USB ड्राइव्ह कनेक्‍ट केला"</string>
     <string name="storage_new_action_browse" msgid="9041125660441839453">"ब्राउझ करा"</string>
@@ -401,9 +401,9 @@
     <string name="location_reporting" msgid="3054236612250063073">"स्थान अहवाल"</string>
     <string name="location_history" msgid="2946680949724247804">"स्थान इतिहास"</string>
     <string name="location_reporting_desc" msgid="6256424995139430823">"Google हे वैशिष्ट्य Google Now आणि Google नकाशे यासारख्या उत्पादनांमध्ये वापरते. स्थान अहवाल चालू करण्यामुळे हे वैशिष्ट्य वापरणार्‍या कोणत्याही Google उत्पादनास आपल्या Google खात्याशी संबंधित आपल्या डिव्हाइसचा सर्वात अलीकडील स्थान डेटा स्टोअर करू आणि वापरू दिला जातो."</string>
-    <string name="location_history_desc" msgid="3861249867748294369">"या खात्यासाठी स्थान इतिहास चालू केला जातो, तेव्हा आपल्या अॅप्सद्वारे वापरण्यासाठी Google आपल्या डिव्हाइसचा स्थान डेटा स्टोअर करू शकते.\n\nउदाहरणार्थ, Google नकाशे आपल्याला दिशानिर्देश देऊ शकते आणि Google Now आपल्याला नियत प्रवास रहदारीविषयी सूचित करू शकते.\n\nआपण कधीही स्थान इतिहास बंद करू शकता, परंतु असे करताना तो हटवला जात नाही. आपला स्थान इतिहास पाहण्यासाठी आणि व्यवस्थापित करण्यासाठी, maps.google.com/locationhistory ला भेट द्या."</string>
+    <string name="location_history_desc" msgid="3861249867748294369">"या खात्यासाठी स्थान इतिहास चालू केला जातो, तेव्हा आपल्या अॅप्सद्वारे वापरण्यासाठी Google आपल्या डिव्हाइसचा स्थान डेटा स्टोअर करू शकते.\n\nउदाहरणार्थ, Google नकाशे तुम्हाला दिशानिर्देश देऊ शकते आणि Google Now तुम्हाला नियत प्रवास रहदारीविषयी सूचित करू शकते.\n\nतुम्ही कधीही स्थान इतिहास बंद करू शकता, परंतु असे करताना तो हटवला जात नाही. तुमचा स्थान इतिहास पाहण्यासाठी आणि व्यवस्थापित करण्यासाठी, maps.google.com/locationhistory ला भेट द्या."</string>
     <string name="delete_location_history_title" msgid="5804802702252280549">"स्थान इतिहास हटवा"</string>
-    <string name="delete_location_history_desc" msgid="4142806827237093397">"हे या Google खात्याकरिता या डिव्हाइसवरून स्टोअर केलेला सर्व स्थान इतिहास हटवेल. आपण हे हटविणे पूर्ववत करू शकता. Google Now सह, काही अॅप्स कार्य करणे थांबवतील."</string>
+    <string name="delete_location_history_desc" msgid="4142806827237093397">"हे या Google खात्याकरिता या डिव्हाइसवरून स्टोअर केलेला सर्व स्थान इतिहास हटवेल. तुम्ही हे हटविणे पूर्ववत करू शकता. Google Now सह, काही अॅप्स कार्य करणे थांबवतील."</string>
     <string name="system_services" msgid="7230571820151215779">"सेवा"</string>
     <string name="accessibility_service_settings" msgid="322888335240595695">"सेवा सेटिंग्ज"</string>
     <string name="accessibility_toggle_high_text_contrast_preference_title" msgid="4970438261747914408">"उच्च तीव्रता मजकूर"</string>
@@ -451,7 +451,7 @@
     <string name="system_accessibility_status" msgid="6150600992901856367">"सक्षम करा"</string>
     <string name="system_accessibility_config" msgid="3216531917536720277">"कॉन्फिगरेशन"</string>
     <string name="system_accessibility_service_on_confirm_title" msgid="8200707259067035823">"<xliff:g id="SERVICE">%1$s</xliff:g> वापरायची?"</string>
-    <string name="system_accessibility_service_on_confirm_desc" msgid="1102358356500113060">"<xliff:g id="SERVICE">%1$s</xliff:g> संकेतशब्दांशिवाय, आपण टाइप करता तो सर्व मजकूर संकलित करू शकते. यात क्रेडिट कार्ड नंबर सारखा वैयक्तिक डेटा समाविष्ट होतो."</string>
+    <string name="system_accessibility_service_on_confirm_desc" msgid="1102358356500113060">"<xliff:g id="SERVICE">%1$s</xliff:g> संकेतशब्दांशिवाय, तुम्ही टाइप करता तो सर्व मजकूर संकलित करू शकते. यात क्रेडिट कार्ड नंबर सारखा वैयक्तिक डेटा समाविष्ट होतो."</string>
     <string name="system_accessibility_service_off_confirm_title" msgid="963297812889694832">"<xliff:g id="SERVICE">%1$s</xliff:g> थांबवायचे?"</string>
     <string name="system_accessibility_service_off_confirm_desc" msgid="6714782635926048678">"ठीक निवडल्याने <xliff:g id="SERVICE">%1$s</xliff:g> थांबेल."</string>
     <string name="system_accessibility_tts_output" msgid="8905477795536714590">"टेक्स्ट टू स्पीच"</string>
@@ -482,7 +482,7 @@
     <string name="system_select_debug_app" msgid="8592021815119618485">"डीबग अॅप निवडा"</string>
     <string name="system_wait_for_debugger" msgid="4007354935701089555">"डीबगरची प्रतीक्षा करा"</string>
     <string name="system_verify_apps_over_usb" msgid="7340861918835270609">"USB वरील अॅप्स सत्यापित करा"</string>
-    <string name="system_desc_verify_apps_over_usb" msgid="6722973410408262393">"हानिकारक वर्तनासाठी ADB/ADT द्वारे इंस्टॉल केलेले अॅप्स तपासा"</string>
+    <string name="system_desc_verify_apps_over_usb" msgid="6722973410408262393">"हानिकारक वर्तनासाठी ADB/ADT द्वारे इंस्टॉल केलेली अॅप्स तपासा"</string>
     <string name="system_wifi_verbose_logging" msgid="3474525355071613242">"वाय-फाय पाल्हाळिक लॉगिंग"</string>
     <string name="system_desc_wifi_verbose_logging" msgid="1559904751873455868">"वाय-फाय पाल्हाळिक लॉगिंग सक्षम करा"</string>
     <string name="system_show_touches" msgid="2179121368437801757">"स्पर्श दर्शवा"</string>
@@ -503,7 +503,7 @@
     <string name="system_show_all_anrs" msgid="3601183436857424712">"सर्व ANR दर्शवा"</string>
     <string name="system_desc_stay_awake" msgid="229588006508356238">"निष्क्रिय करा ला अक्षम करा"</string>
     <string name="system_desc_hdcp_checking" msgid="6879753401058891183">"केवळ DRM सामग्रीसाठी वापरा"</string>
-    <string name="system_desc_hdmi_optimization" msgid="4400727262768931480">"अधिकतम रिझोल्यूशनसाठी किंवा अधिकतम फ्रेमरेटसाठी प्रदर्शन ऑप्टिमाइझ करा. हे केवळ Ultra HD प्रदर्शने प्रभावित करते. हे सेटिंग बदलण्याने आपले डिव्हाइस रीस्टार्ट होईल."</string>
+    <string name="system_desc_hdmi_optimization" msgid="4400727262768931480">"अधिकतम रिझोल्यूशनसाठी किंवा अधिकतम फ्रेमरेटसाठी प्रदर्शन ऑप्टिमाइझ करा. हे केवळ Ultra HD प्रदर्शने प्रभावित करते. हे सेटिंग बदलण्याने तुमचे डिव्हाइस रीस्टार्ट होईल."</string>
     <string name="system_desc_bt_hci_log" msgid="4219265203570411773">"ब्लूटूथ HCI स्नूप लॉगिंग सक्षम करा"</string>
     <string name="system_desc_usb_debugging" msgid="5168830368191788731">"USB कनेक्‍ट केल्‍यावर मोड डीबग करा"</string>
     <string name="system_desc_wait_for_debugger" msgid="2342968959255039368">"डीबग केलेला अॅप्लिकेशन अंमलात आणण्यापूर्वी डीबगर संलग्न करण्याची प्रतीक्षा करतो"</string>
@@ -523,7 +523,7 @@
     <string name="security_unknown_sources_title" msgid="769762478057744545">"अज्ञात स्त्रोत"</string>
     <string name="security_unknown_sources_desc" msgid="7197053279914817124">"Play Store च्या व्यतिरिक्त इतर स्रोतांवरील अॅप्सच्या स्थापनेस अनुमती द्या"</string>
     <string name="security_unknown_sources_confirm_title" msgid="1249752174105198821">"अज्ञात स्रोतांना अनुमती द्या"</string>
-    <string name="security_unknown_sources_confirm_desc" msgid="7167350583097191925">"आपले डिव्‍हाइस आणि वैयक्तिक डेटा अज्ञात स्त्रोतांवरील अ‍ॅप्‍सद्वारे होणार्‍या आक्रमणांना अधिक असुरक्षित असतात. आपण सहमती देता की हे अ‍ॅप्स वापरल्यामुळे आपल्या डिव्‍हाइसच्या होणार्‍या कोणत्याही नुकसानास किंवा डेटाच्या हानीस आपण पूर्णपणे जबाबदार आहात."</string>
+    <string name="security_unknown_sources_confirm_desc" msgid="7167350583097191925">"तुमचे डिव्‍हाइस आणि वैयक्तिक डेटा अज्ञात स्त्रोतांवरील अ‍ॅप्‍सद्वारे होणार्‍या आक्रमणांना अधिक असुरक्षित असतात. तुम्ही सहमती देता की हे अ‍ॅप्स वापरल्यामुळे आपल्या डिव्‍हाइसच्या होणार्‍या कोणत्याही नुकसानास किंवा डेटाच्या हानीस तुम्ही पूर्णपणे जबाबदार आहात."</string>
     <string name="security_verify_apps_title" msgid="5787946958632254988">"अॅप्स सत्यापित करा"</string>
     <string name="security_verify_apps_desc" msgid="1456317893117818733">"ज्यामुळे हानी होऊ शकते अशा अॅप्सच्या स्थापनेस अनुमती देऊ नका किंवा त्यापूर्वी चेतावणी द्या"</string>
     <string name="system_hdcp_checking_never" msgid="1758043078676276985">"कधीही नाही"</string>
@@ -567,7 +567,7 @@
     <string name="autofill_none" msgid="2426888642367790651">"काहीही नाही"</string>
     <string name="autofill_confirmation_message" msgid="9068009043054751287">"&lt;b&gt;तुम्हाला या अॅपवर विश्वास असल्याचे सुनिश्चित करा&lt;/b&gt; &lt;br/&gt; &lt;br/&gt; &lt;xliff:g id=app_name example=Google Autofill&gt;%1$s&lt;/xliff:g&gt; काय ऑटोफिल होऊ शकते हे ठरवण्यासाठी तुमच्या स्क्रीनवर काय आहे याचा वापर करते."</string>
     <string name="computing_size" msgid="2955133568424704042">"गणना करत आहे…"</string>
-    <string name="title_select_wifi_network" msgid="1038871400014761952">"आपले वाय-फाय नेटवर्क निवडा"</string>
+    <string name="title_select_wifi_network" msgid="1038871400014761952">"तुमचे वाय-फाय नेटवर्क निवडा"</string>
     <string name="accessories_wifi_display_rename_device" msgid="7416728299196887186">"पुनर्नामित करा"</string>
     <string name="accessories_wifi_display_enable" msgid="4779506305656453262">"वाय-फाय प्रदर्शन"</string>
     <string name="accessories_wifi_display_pin_required" msgid="3242645267864813273">"पिन आवश्यक"</string>
@@ -638,7 +638,7 @@
     <string name="pin_enter_new_pin" msgid="4788218421265924211">"नवीन पिन सेट करा"</string>
     <string name="pin_enter_again" msgid="1510456697040826496">"नवीन पिन पुन्हा प्रविष्‍ट करा"</string>
     <string name="pin_enter_old_pin" msgid="851029058127630869">"जुना पिन प्रविष्‍ट करा"</string>
-    <string name="pin_enter_wrong_seconds" msgid="7541208202846169755">"आपण चुकीचा पिन 5 वेळा प्रविष्‍ट केला.\n<xliff:g id="RELATIVE_TIME_SPAN">%1$d</xliff:g> सेकंदांनी पुन्हा प्रयत्न करा."</string>
+    <string name="pin_enter_wrong_seconds" msgid="7541208202846169755">"तुम्ही चुकीचा पिन 5 वेळा प्रविष्‍ट केला.\n<xliff:g id="RELATIVE_TIME_SPAN">%1$d</xliff:g> सेकंदांनी पुन्हा प्रयत्न करा."</string>
     <string name="pin_toast_wrong" msgid="4628358195224633248">"चुकीचा पिन, पुन्हा प्रयत्न करा"</string>
     <string name="pin_toast_not_match" msgid="5886041789737632996">"पुन्हा प्रयत्न करा, पिन जुळत नाही"</string>
     <string name="wifi_setup_input_password" msgid="1596168874369038536">"<xliff:g id="SSID">%1$s</xliff:g> साठी पासवर्ड प्रविष्‍ट करा"</string>
@@ -647,14 +647,14 @@
     <string name="device_apps_app_management_version" msgid="2267065043941006486">"आवृत्ती <xliff:g id="APP_VERSION">%1$s</xliff:g>"</string>
     <string name="device_apps_app_management_open" msgid="8009823973422120839">"उघडा"</string>
     <string name="device_apps_app_management_force_stop" msgid="9195433885986760867">"सक्तीने थांबवा"</string>
-    <string name="device_apps_app_management_force_stop_desc" msgid="3616902338964109665">"आपण अ‍ॅपला सक्तीेने थांबविल्‍यास, ते कदाचित गैरवर्तन असू शकते."</string>
+    <string name="device_apps_app_management_force_stop_desc" msgid="3616902338964109665">"तुम्ही अ‍ॅपला सक्तीेने थांबविल्‍यास, ते कदाचित गैरवर्तन असू शकते."</string>
     <string name="device_apps_app_management_uninstall" msgid="531427430753359746">"अनइंस्टॉल करा"</string>
     <string name="device_apps_app_management_uninstall_updates" msgid="5151008748970915803">"अपडेट अनइंस्टॉल करा"</string>
     <string name="device_apps_app_management_uninstall_updates_desc" msgid="9165160845303755321">"या Android सिस्टम अॅप वरील सर्व अपडेट अनइंस्टॉल केली जातील."</string>
     <string name="device_apps_app_management_disable" msgid="6136528600052941589">"अक्षम करा"</string>
-    <string name="device_apps_app_management_disable_desc" msgid="1295507373252301108">"आपण हा अॅप अक्षम करू इच्छिता?"</string>
+    <string name="device_apps_app_management_disable_desc" msgid="1295507373252301108">"तुम्ही हा अॅप अक्षम करू इच्छिता?"</string>
     <string name="device_apps_app_management_enable" msgid="3246823119203444608">"सक्षम करा"</string>
-    <string name="device_apps_app_management_enable_desc" msgid="7490219375247065314">"आपण हा अॅप सक्षम करू इच्छिता?"</string>
+    <string name="device_apps_app_management_enable_desc" msgid="7490219375247065314">"तुम्ही हा अॅप सक्षम करू इच्छिता?"</string>
     <string name="device_apps_app_management_storage_used" msgid="4611351576179140634">"वापरलेले संचयन"</string>
     <string name="device_apps_app_management_storage_used_desc" msgid="1529188874944834661">"<xliff:g id="VOLUME">%2$s</xliff:g> मध्ये <xliff:g id="SIZE">%1$s</xliff:g> वापरले"</string>
     <string name="device_apps_app_management_clear_data" msgid="4439114679323274127">"डेटा साफ करा"</string>
@@ -680,13 +680,13 @@
     <string name="device_daydreams_screen_off_description" msgid="4181262365345306243">"निष्क्रियतेच्या या कालावधीनंतर स्क्रीन बंद होते."</string>
     <string name="backup_configure_account_default_summary" msgid="5619226302157702967">"कोणतेही खाते सध्‍या बॅकअप घेतलेला डेटा स्टोअर करीत नाही"</string>
     <string name="backup_erase_dialog_title" msgid="3720907811680819429"></string>
-    <string name="backup_erase_dialog_message" msgid="2608496013975068701">"आपले वाय-फाय पासवर्ड, बुकमार्क, अन्य सेटिंग्ज आणि अॅप डेटा चा बॅक अप घेणे बंद करायचे आणि Google सर्व्हरवरील सर्व प्रती मिटवायच्या?"</string>
+    <string name="backup_erase_dialog_message" msgid="2608496013975068701">"तुमचे वाय-फाय पासवर्ड, बुकमार्क, अन्य सेटिंग्ज आणि अॅप डेटा चा बॅक अप घेणे बंद करायचे आणि Google सर्व्हरवरील सर्व प्रती मिटवायच्या?"</string>
     <string name="privacy_backup_data" msgid="1796662583830120812">"माझ्या डेटाचा बॅकअप घ्या"</string>
     <string name="privacy_backup_account" msgid="9059939814827165020">"बॅक अप खाते"</string>
     <string name="privacy_automatic_restore" msgid="1409885318093199628">"स्‍वयंचलित पुनर्संचयन"</string>
     <string name="factory_reset_device" msgid="7422459553658815011">"डिव्‍हाइस रीसेट करा"</string>
-    <string name="factory_reset_description" msgid="5381170999141355891">"हे आपल्‍या डिव्‍हाइसच्‍या अंतर्गत संचयनावरील सर्व डेटा याच्या समावेशासह मिटवेल: आपले Google खाते, सिस्‍टीम आणि अ‍ॅप डेटा आणि सेटिंग्‍ज, आणि डाउनलोड केलेले अ‍ॅप्‍स"</string>
-    <string name="confirm_factory_reset_description" msgid="1049842658625287303">"तुमची सर्व वैयक्तिक माहिती आणि डाउनलोड केलेले अ‍ॅप्स मिटयावचे? आपण ही क्रिया उलट करू शकत नाही!"</string>
+    <string name="factory_reset_description" msgid="5381170999141355891">"हे आपल्‍या डिव्‍हाइसच्‍या अंतर्गत संचयनावरील सर्व डेटा याच्या समावेशासह मिटवेल: तुमचे Google खाते, सिस्‍टीम आणि अ‍ॅप डेटा आणि सेटिंग्‍ज, आणि डाउनलोड केलेले अ‍ॅप्‍स"</string>
+    <string name="confirm_factory_reset_description" msgid="1049842658625287303">"तुमची सर्व वैयक्तिक माहिती आणि डाउनलोड केलेले अ‍ॅप्स मिटयावचे? तुम्ही ही क्रिया उलट करू शकत नाही!"</string>
     <string name="confirm_factory_reset_device" msgid="8923287482399582268">"प्रत्येकगोष्ट मिटवा"</string>
     <string name="select_device_name_title" msgid="5165800500130943604">"आपल्‍या <xliff:g id="DEVICEMODEL">%1$s</xliff:g> साठी एक नाव निवडा"</string>
     <string name="select_device_name_description" msgid="4939767901420114190">"तुमचे डिव्‍हाइस इतर डिव्‍हाइसवरून कास्‍ट होत असताना किंवा कनेक्‍ट होत असताना ओळखता येण्‍यासाठी मदत होण्‍याकरता त्‍यास नाव द्या."</string>
@@ -711,8 +711,8 @@
     <string name="choose_application" msgid="5662960665577780353">"अॅप्लिकेशन निवडा"</string>
     <string name="experimental_preference" msgid="1137673985823581797">"(प्रायोगिक)"</string>
     <string name="reboot_safemode_action" msgid="8129550438717110406">"सुरक्षित मोडमध्ये रीबूट करा"</string>
-    <string name="reboot_safemode_confirm" msgid="3329475617645192086">"आपण सुरक्षित मोडमध्ये रीबूट करू इच्छिता?"</string>
-    <string name="reboot_safemode_desc" msgid="5031024337104679492">"हे आपण इंस्टॉल केलेल्या सर्व तृतीय पक्ष अनुप्रयोगांना अक्षम करेल. आपण पुन्हा रीबूट कराल तेव्हा त्यांना पुनर्संचयित केले जाईल."</string>
+    <string name="reboot_safemode_confirm" msgid="3329475617645192086">"तुम्ही सुरक्षित मोडमध्ये रीबूट करू इच्छिता?"</string>
+    <string name="reboot_safemode_desc" msgid="5031024337104679492">"हे तुम्ही इंस्टॉल केलेल्या सर्व तृतीय पक्ष अनुप्रयोगांना अक्षम करेल. तुम्ही पुन्हा रीबूट कराल तेव्हा त्यांना पुनर्संचयित केले जाईल."</string>
     <string name="capturing_bugreport" msgid="3291125403411239592">"दोष अहवाल कॅप्चर करत आहे"</string>
     <string name="available_virtual_keyboard_category" msgid="6566257204210337817">"उपलब्ध व्हर्च्युअल कीबोर्ड"</string>
     <string name="manage_keyboards" msgid="1905379263117016795">"कीबोर्ड व्यवस्थापित करा"</string>
diff --git a/Settings/res/values-nb/strings.xml b/Settings/res/values-nb/strings.xml
index f370a977..e3c13b21 100644
--- a/Settings/res/values-nb/strings.xml
+++ b/Settings/res/values-nb/strings.xml
@@ -78,7 +78,7 @@
     <string name="all_apps_show_system_apps" msgid="4648447904721171884">"Vis systemapper"</string>
     <string name="all_apps_installed" msgid="54800379871111518">"Installerte apper"</string>
     <string name="all_apps_other" msgid="8297097642727891911">"Systemapper"</string>
-    <string name="all_apps_disabled" msgid="2016067533586741345">"Avslåtte apper"</string>
+    <string name="all_apps_disabled" msgid="2016067533586741345">"Deaktiverte apper"</string>
     <string name="device_daydream" msgid="3354366974209179614">"Skjermsparer"</string>
     <string name="device_display" msgid="101455384386372747">"Skjerm"</string>
     <string name="device_sound" msgid="9214182566287657481">"Lyd"</string>
diff --git a/Settings/res/values-ne/strings.xml b/Settings/res/values-ne/strings.xml
index 74c07a6b..f9a1956d 100644
--- a/Settings/res/values-ne/strings.xml
+++ b/Settings/res/values-ne/strings.xml
@@ -650,7 +650,7 @@
     <string name="device_apps_app_management_force_stop_desc" msgid="3616902338964109665">"तपाईंले अनुप्रयोग बन्द गर्न बल गर्नुभयो भने, यसले चाहेअनुसार काम नगर्न सक्छ।"</string>
     <string name="device_apps_app_management_uninstall" msgid="531427430753359746">"स्थापना हटाउनुहोस्"</string>
     <string name="device_apps_app_management_uninstall_updates" msgid="5151008748970915803">"अद्यावधिकहरूको स्थापना रद्द गर्नुहोस्"</string>
-    <string name="device_apps_app_management_uninstall_updates_desc" msgid="9165160845303755321">"यस Android प्रणाली अनुप्रयोगमा भएका सम्पूर्ण अद्यावधिकहरूको स्थापना हटाइनेछ।"</string>
+    <string name="device_apps_app_management_uninstall_updates_desc" msgid="9165160845303755321">"यस Android प्रणाली अनुप्रयोगमा भएका सम्पूर्ण अद्यावधिकहरूको स्थापना हटाइने छ।"</string>
     <string name="device_apps_app_management_disable" msgid="6136528600052941589">"अक्षम गर्नुहोस्"</string>
     <string name="device_apps_app_management_disable_desc" msgid="1295507373252301108">"तपाईं यो अनुप्रयोग अक्षम गर्न चाहनुहुन्छ?"</string>
     <string name="device_apps_app_management_enable" msgid="3246823119203444608">"सक्षम गर्नुहोस्"</string>
diff --git a/Settings/res/values-pl/strings.xml b/Settings/res/values-pl/strings.xml
index 24898d0a..227fa017 100644
--- a/Settings/res/values-pl/strings.xml
+++ b/Settings/res/values-pl/strings.xml
@@ -350,7 +350,7 @@
     <string name="storage_wizard_back_up_apps_space_available" msgid="7209210725057470544">"Dostępne: <xliff:g id="SIZE">%1$s</xliff:g>"</string>
     <string name="storage_wizard_eject_private_title" msgid="2844023177159233031">"Odłącz pamięć urządzenia"</string>
     <string name="storage_wizard_eject_private_description" msgid="8966427719795472538">"Gdy odłączysz ten nośnik pamięci, znajdujące się na nim aplikacje przestaną działać. Ten dysk USB jest sformatowany w taki sposób, że będzie działać tylko na tym urządzeniu. Nie będzie działać na żadnych innych urządzeniach."</string>
-    <string name="storage_wizard_eject_progress_title" msgid="8823133087157003820">"Wyuwam dysk <xliff:g id="NAME">%1$s</xliff:g>..."</string>
+    <string name="storage_wizard_eject_progress_title" msgid="8823133087157003820">"Wysuwam dysk <xliff:g id="NAME">%1$s</xliff:g>..."</string>
     <string name="storage_wizard_move_app_title" msgid="37057801779891287">"Wykorzystane miejsce"</string>
     <string name="storage_wizard_move_app_progress_title" msgid="7807062446517628864">"Przenoszę <xliff:g id="NAME">%1$s</xliff:g>…"</string>
     <string name="storage_wizard_move_app_progress_description" msgid="3693104566187904724">"Nie wyjmuj dysku w trakcie przenoszenia danych.\nAplikacja <xliff:g id="APPNAME">%1$s</xliff:g> na tym urządzeniu będzie niedostępna, dopóki przenoszenie się nie zakończy."</string>
diff --git a/Settings/res/values-pt-rBR/strings.xml b/Settings/res/values-pt-rBR/strings.xml
index a837df31..602931cb 100644
--- a/Settings/res/values-pt-rBR/strings.xml
+++ b/Settings/res/values-pt-rBR/strings.xml
@@ -185,7 +185,7 @@
     <string name="device_status_title" msgid="1629253652287243543">"Status"</string>
     <string name="battery_status_title" msgid="2557264957661087826">"Status da bateria"</string>
     <string name="battery_level_title" msgid="100046129713781009">"Nível da bateria"</string>
-    <string name="sim_status_title" msgid="498929467188844611">"Status do SIM"</string>
+    <string name="sim_status_title" msgid="498929467188844611">"Status do chip"</string>
     <string name="imei_information_title" msgid="5208860549282877136">"Informações de IMEI"</string>
     <string name="status_bt_address" msgid="1571721865060122393">"Endereço Bluetooth"</string>
     <string name="status_up_time" msgid="1674116490191446110">"Tempo de atividade"</string>
diff --git a/Settings/res/values-pt-rPT/strings.xml b/Settings/res/values-pt-rPT/strings.xml
index 498d80f3..c9eb472c 100644
--- a/Settings/res/values-pt-rPT/strings.xml
+++ b/Settings/res/values-pt-rPT/strings.xml
@@ -49,7 +49,7 @@
     </plurals>
     <string name="add_account" msgid="4098060223200619258">"Adicionar conta"</string>
     <string name="account_header_remove_account" msgid="3288957118207682685">"Remover conta"</string>
-    <string name="account_sync" msgid="773808124331714965">"Escolher aplic. sincronizadas"</string>
+    <string name="account_sync" msgid="773808124331714965">"Escolher app sincronizadas"</string>
     <string name="sync_now" msgid="8732244492880016471">"Sincronizar agora"</string>
     <string name="sync_in_progress" msgid="1330721038775646022">"A sincronizar..."</string>
     <string name="last_synced" msgid="6178494892357077274">"Última sincronização: <xliff:g id="TIME">%1s</xliff:g>"</string>
@@ -102,7 +102,7 @@
     <string name="surround_sound_none_summary" msgid="4295042972785042496">"Nenhum: nunca utilizar o som surround"</string>
     <string name="surround_sound_manual_summary" msgid="9208540024024014501">"Manual: selecionar os formatos a utilizar"</string>
     <string name="device_storage_clear_cache_title" msgid="3365635664766067643">"Pretende limpar os dados em cache?"</string>
-    <string name="device_storage_clear_cache_message" msgid="6495570470005696210">"Isto limpa os dados em cache de todas as aplic."</string>
+    <string name="device_storage_clear_cache_message" msgid="6495570470005696210">"Isto limpa os dados em cache de todas as app"</string>
     <string name="accessories_add" msgid="1347802656349218830">"Adicionar acessório"</string>
     <string name="accessory_state_pairing" msgid="7501989679637780963">"A sincronizar..."</string>
     <string name="accessory_state_connecting" msgid="8777884957977608529">"A ligar..."</string>
@@ -512,7 +512,7 @@
     <string name="system_desc_show_hardware_layer" msgid="642854466995400529">"Destacar camadas de hardware a verde ao atualizar"</string>
     <string name="system_desc_show_gpu_overdraw" msgid="8204945204510618865">"Melhor para pior: azul, verde, verm. claro, verm."</string>
     <string name="system_desc_show_surface_updates" msgid="8177734845253807688">"Destacar todas as superf. de janelas ao atualizar"</string>
-    <string name="system_desc_strict_mode_enabled" msgid="4647420860235689510">"Piscar qd aplic. fazem oper. prol. tópico princ."</string>
+    <string name="system_desc_strict_mode_enabled" msgid="4647420860235689510">"Piscar qd app fazem oper. prol. tópico princ."</string>
     <string name="system_desc_profile_gpu_rendering" msgid="2836283129486718887">"Medir tempo de conv. em adb shell dumpsys gfxinfo"</string>
     <string name="action_on_title" msgid="6852389884236870765">"Ativar"</string>
     <string name="action_off_title" msgid="3157728958166559966">"Desativar"</string>
@@ -704,7 +704,7 @@
     <string name="change_setting" msgid="2047402388786162246">"Alterar"</string>
     <string name="keep_settings" msgid="6798327196645732517">"Não alterar"</string>
     <string name="apps_permissions" msgid="1283592091618660965">"Autorizações"</string>
-    <string name="device_apps_permissions" msgid="2966387603529725982">"Autoriz. da aplic."</string>
+    <string name="device_apps_permissions" msgid="2966387603529725982">"Autoriz. da app"</string>
     <string name="app_permissions_group_summary" msgid="7906834786705287901">"<xliff:g id="COUNT_0">%1$d</xliff:g> de <xliff:g id="COUNT_1">%2$d</xliff:g> aplicações autorizadas"</string>
     <string name="bluetooth_permission_request" msgid="6649813878683015301">"Pedido de autorização de Bluetooth"</string>
     <string name="security_patch" msgid="2131617390189164481">"Nível do patch de segurança do Android"</string>
diff --git a/Settings/res/values-pt/strings.xml b/Settings/res/values-pt/strings.xml
index a837df31..602931cb 100644
--- a/Settings/res/values-pt/strings.xml
+++ b/Settings/res/values-pt/strings.xml
@@ -185,7 +185,7 @@
     <string name="device_status_title" msgid="1629253652287243543">"Status"</string>
     <string name="battery_status_title" msgid="2557264957661087826">"Status da bateria"</string>
     <string name="battery_level_title" msgid="100046129713781009">"Nível da bateria"</string>
-    <string name="sim_status_title" msgid="498929467188844611">"Status do SIM"</string>
+    <string name="sim_status_title" msgid="498929467188844611">"Status do chip"</string>
     <string name="imei_information_title" msgid="5208860549282877136">"Informações de IMEI"</string>
     <string name="status_bt_address" msgid="1571721865060122393">"Endereço Bluetooth"</string>
     <string name="status_up_time" msgid="1674116490191446110">"Tempo de atividade"</string>
diff --git a/Settings/res/values-sv/strings.xml b/Settings/res/values-sv/strings.xml
index 389fde36..cb841dae 100644
--- a/Settings/res/values-sv/strings.xml
+++ b/Settings/res/values-sv/strings.xml
@@ -492,8 +492,8 @@
     <string name="system_show_hardware_layer" msgid="7301942527058509453">"Visa maskinvaruskikt"</string>
     <string name="system_show_gpu_overdraw" msgid="707861254106800199">"Visa GPU-överritning"</string>
     <string name="system_show_surface_updates" msgid="8878834675712192327">"Visa ytuppdateringar"</string>
-    <string name="system_window_animation_scale" msgid="3213952676672537856">"Skala för fönsteranimering"</string>
-    <string name="system_transition_animation_scale" msgid="2964817709721784757">"Skala för övergångsanimering"</string>
+    <string name="system_window_animation_scale" msgid="3213952676672537856">"Skala – fönsteranimering"</string>
+    <string name="system_transition_animation_scale" msgid="2964817709721784757">"Skala – övergångsanimering"</string>
     <string name="system_animator_duration_scale" msgid="7413512458963984521">"Längdskala för Animator"</string>
     <string name="system_strict_mode_enabled" msgid="5728680397806673270">"Strikt läge aktiverat"</string>
     <string name="system_profile_gpu_rendering" msgid="3438771015102629799">"Profilens GPU-rendering"</string>
diff --git a/Settings/res/values-sw/strings.xml b/Settings/res/values-sw/strings.xml
index f25dfb5d..b6cef4b7 100644
--- a/Settings/res/values-sw/strings.xml
+++ b/Settings/res/values-sw/strings.xml
@@ -142,7 +142,7 @@
     <string name="system_speech" msgid="8128688499687306982">"Usemi"</string>
     <string name="system_inputs" msgid="2413907183796651102">"Vifaa vya kuingiza sauti"</string>
     <string name="system_inputs_devices" msgid="8700905419497948327">"Vifaa vya kuingiza sauti"</string>
-    <string name="system_home_theater_control" msgid="2053198603790384368">"Udhibiti wa vifaa vya burudani vya nyumbani"</string>
+    <string name="system_home_theater_control" msgid="2053198603790384368">"Kudhibiti vifaa vya burudani vya nyumbani"</string>
     <string name="system_accessibility" msgid="6215431399759498005">"Ufikiaji"</string>
     <string name="system_developer_options" msgid="3237985523474863638">"Chaguo za wasanidi programu"</string>
     <string name="accessibility_none" msgid="3441648940134070004">"Hakuna"</string>
@@ -649,7 +649,7 @@
     <string name="device_apps_app_management_force_stop" msgid="9195433885986760867">"Ilazimishe ikome"</string>
     <string name="device_apps_app_management_force_stop_desc" msgid="3616902338964109665">"Ukilazimisha programu kuzima, inaweza kutofanya vizuri."</string>
     <string name="device_apps_app_management_uninstall" msgid="531427430753359746">"Ondoa"</string>
-    <string name="device_apps_app_management_uninstall_updates" msgid="5151008748970915803">"Sanidua masasisho"</string>
+    <string name="device_apps_app_management_uninstall_updates" msgid="5151008748970915803">"Ondoa masasisho"</string>
     <string name="device_apps_app_management_uninstall_updates_desc" msgid="9165160845303755321">"Masasisho yote ya programu hii ya mfumo wa Android yataondolewa."</string>
     <string name="device_apps_app_management_disable" msgid="6136528600052941589">"Zima"</string>
     <string name="device_apps_app_management_disable_desc" msgid="1295507373252301108">"Unataka kuzima programu hii?"</string>
diff --git a/Settings/res/values-ta/strings.xml b/Settings/res/values-ta/strings.xml
index 415d6b96..3eb63c60 100644
--- a/Settings/res/values-ta/strings.xml
+++ b/Settings/res/values-ta/strings.xml
@@ -492,8 +492,8 @@
     <string name="system_show_hardware_layer" msgid="7301942527058509453">"வன்பொருள் லேயரைக் காட்டு"</string>
     <string name="system_show_gpu_overdraw" msgid="707861254106800199">"GPU ஓவர்டிராவைக் காட்டு"</string>
     <string name="system_show_surface_updates" msgid="8878834675712192327">"மேலோட்ட புதுப்பிப்புகளைக் காட்டு"</string>
-    <string name="system_window_animation_scale" msgid="3213952676672537856">"சாளர அனிமேஷன் அளவு"</string>
-    <string name="system_transition_animation_scale" msgid="2964817709721784757">"அனிமேஷன் மாற்றத்தின் அளவு"</string>
+    <string name="system_window_animation_scale" msgid="3213952676672537856">"சாளர அனிமேஷன் வேகம்"</string>
+    <string name="system_transition_animation_scale" msgid="2964817709721784757">"அனிமேஷன் மாற்றத்தின் வேகம்"</string>
     <string name="system_animator_duration_scale" msgid="7413512458963984521">"அனிமேட்டர் கால அளவு"</string>
     <string name="system_strict_mode_enabled" msgid="5728680397806673270">"நிலையான பயன்முறை இயக்கப்பட்டது"</string>
     <string name="system_profile_gpu_rendering" msgid="3438771015102629799">"சுயவிவர GPU வழங்கல்"</string>
@@ -538,12 +538,12 @@
     <string name="enable_opengl_traces_none" msgid="6074804146602749420">"ஏதுமில்லை"</string>
   <string-array name="animation_scale_entries">
     <item msgid="7477134110460336436">"அனிமேஷனை முடக்கு"</item>
-    <item msgid="8327257151534006014">"அனிமேஷன் அளவு .5x"</item>
-    <item msgid="2105786719544636780">"அனிமேஷன் அளவு 1x"</item>
-    <item msgid="5290554136790336403">"அனிமேஷன் அளவு 1.5x"</item>
-    <item msgid="1232929590838082483">"அனிமேஷன் அளவு 2x"</item>
-    <item msgid="1889239009419613796">"அனிமேஷன் அளவு 5x"</item>
-    <item msgid="7268894934484547297">"அனிமேஷன் அளவு 10x"</item>
+    <item msgid="8327257151534006014">"அனிமேஷன் வேகம் .5x"</item>
+    <item msgid="2105786719544636780">"அனிமேஷன் வேகம் 1x"</item>
+    <item msgid="5290554136790336403">"அனிமேஷன் வேகம் 1.5x"</item>
+    <item msgid="1232929590838082483">"அனிமேஷன் வேகம் 2x"</item>
+    <item msgid="1889239009419613796">"அனிமேஷன் வேகம் 5x"</item>
+    <item msgid="7268894934484547297">"அனிமேஷன் வேகம் 10x"</item>
   </string-array>
     <string name="track_frame_time_off" msgid="5050360262132715904">"முடக்கு"</string>
     <string name="track_frame_time_bars" msgid="4470692840177888414">"திரையைப் பட்டிகளாக இயக்கு"</string>
@@ -719,7 +719,7 @@
     <string name="app_permission_summary_allowed" msgid="1618833359547333078">"அனுமதிக்கப்பட்டது"</string>
     <string name="app_permission_summary_not_allowed" msgid="3667663331476577913">"அனுமதிக்கப்படவில்லை"</string>
     <string name="usage_access" msgid="1914242907684452328">"உபயோக அணுகல்"</string>
-    <string name="usage_access_description" msgid="6290001769893747651">"உபயோக அணுகலானது நீங்கள் பயன்படுத்தும் பிற பயன்பாடுகள், அவற்றை எவ்வளவு அடிக்கடிப் பயன்படுத்துகிறீர்கள் என்பதையும், அத்துடன் தொலைத்தொடர்பு நிறுவனம், மொழி அமைப்புகள், பிற விவரங்களையும் கண்காணிக்க, பயன்பாட்டை அனுமதிக்கும்."</string>
+    <string name="usage_access_description" msgid="6290001769893747651">"உபயோக அணுகலானது, நீங்கள் வேறு எந்தெந்த ஆப்ஸை எவ்வளவு அடிக்கடி பயன்படுத்துகிறீர்கள் என்று அறியும் அனுமதியை ஒரு ஆப்ஸுக்கு வழங்குகிறது. உங்கள் மொபைல் நிறுவனம், மொழி அமைப்புகள் மற்றும் பிற விவரங்களையும் அறிந்து கொள்ளும் அனுமதியும் இதில் அடங்கும்."</string>
     <string name="high_power_apps" msgid="7796096221999495077">"பேட்டரி ஆற்றலை மேம்படுத்துதல்"</string>
     <string name="high_power_apps_description" msgid="1990016812115792282">"பேட்டரியைக் குறைவாகப் பயன்படுத்த, ஆப்ஸின் ஆற்றல் உபயோகத்தை மேம்படுத்துதல்"</string>
     <string name="high_power_apps_empty" msgid="1355906551507153470">"பேட்டரியைக் குறைவாகப் பயன்படுத்துவதற்காக, ஆற்றல் மேம்படுத்தப்பட வேண்டிய ஆப்ஸ் எதுவும் இல்லை"</string>
diff --git a/Settings/res/values-th/strings.xml b/Settings/res/values-th/strings.xml
index e10e72c6..d6b0570f 100644
--- a/Settings/res/values-th/strings.xml
+++ b/Settings/res/values-th/strings.xml
@@ -364,7 +364,7 @@
     <string name="invalid_location" msgid="5269044848535157307">"ตำแหน่งที่ติดตั้งไม่ถูกต้อง"</string>
     <string name="system_package" msgid="9200255718634512790">"ติดตั้งการอัปเดตระบบบนสื่อภายนอกไม่ได้"</string>
     <string name="move_error_device_admin" msgid="2060627704270836951">"ไม่สามารถติดตั้งโปรแกรมควบคุมอุปกรณ์บนสื่อภายนอก"</string>
-    <string name="learn_more_action" msgid="7015991954404958779">"เรียนรู้เพิ่มเติม"</string>
+    <string name="learn_more_action" msgid="7015991954404958779">"ดูข้อมูลเพิ่มเติม"</string>
     <string name="system_date" msgid="2767108116769558081">"วันที่"</string>
     <string name="system_time" msgid="6726022312605226285">"เวลา"</string>
     <string name="system_set_date" msgid="2437586307768561116">"ตั้งวันที่"</string>
diff --git a/Settings/res/values-tr/strings.xml b/Settings/res/values-tr/strings.xml
index 8a3ce901..6a05e2b9 100644
--- a/Settings/res/values-tr/strings.xml
+++ b/Settings/res/values-tr/strings.xml
@@ -730,7 +730,7 @@
     <string name="no_notification_listeners" msgid="4583789599075326886">"Yüklü hiçbir uygulama, bildirim erişimi isteğinde bulunmadı."</string>
     <string name="notification_listener_security_warning_summary" msgid="3647205635922300578">"Bu uygulamalar kişi adları ve gelen mesajların metni gibi kişisel bilgiler de dahil olmak üzere tüm bildirimleri okuyabilecek. Ayrıca, bildirimleri kapatabilecek veya bildirime ait eylem düğmelerini tetikleyebilecek."</string>
     <string name="system_alert_window_settings" msgid="2069031369694448339">"Diğer uygulamaların üzerinde göster"</string>
-    <string name="permit_draw_overlay" msgid="2400084757761408906">"Diğer uyg. üstünde gösterime izin ver"</string>
+    <string name="permit_draw_overlay" msgid="2400084757761408906">"Diğer uyg. üzerinde gösterime izin ver"</string>
     <string name="allow_overlay_description" msgid="1946279120428813474">"Bir uygulamanın, kullandığınız diğer uygulamaların üzerinde gösterilmesine izin verir. Bu durum, söz konusu uygulamaları kullanmanızı etkileyebilir veya görünüşlerini ya da davranışlarını değiştirebilir."</string>
     <string name="write_system_settings" msgid="2730923656857824907">"Sistem ayarlarını değiştir"</string>
     <string name="write_settings_title" msgid="8750406725746598710">"Sistem ayarlarını değiştirebilir"</string>
diff --git a/Settings/res/values-uz/strings.xml b/Settings/res/values-uz/strings.xml
index ca87635d..a7779ccf 100644
--- a/Settings/res/values-uz/strings.xml
+++ b/Settings/res/values-uz/strings.xml
@@ -76,7 +76,7 @@
     <string name="apps_permissions_category_title" msgid="4476837368840442313">"Ruxsatlar"</string>
     <string name="all_apps_title" msgid="5330174847732759402">"Barcha ilovalar"</string>
     <string name="all_apps_show_system_apps" msgid="4648447904721171884">"Tizim ilovalarini ko‘rsatish"</string>
-    <string name="all_apps_installed" msgid="54800379871111518">"O‘rnatilgan ilovalar"</string>
+    <string name="all_apps_installed" msgid="54800379871111518">"Oʻrnatilgan ilovalar"</string>
     <string name="all_apps_other" msgid="8297097642727891911">"Tizim ilovalari"</string>
     <string name="all_apps_disabled" msgid="2016067533586741345">"Nofaol ilovalar"</string>
     <string name="device_daydream" msgid="3354366974209179614">"Ekran lavhasi"</string>
@@ -739,7 +739,7 @@
     <string name="write_settings_off" msgid="6219072254626503312">"Yo‘q"</string>
     <string name="picture_in_picture_title" msgid="7738692876349058">"Tasvir ustida tasvir"</string>
     <string name="picture_in_picture_app_detail_switch" msgid="8063965798321494391">"Tasvir ustida tasvir rejimiga ruxsat berish"</string>
-    <string name="picture_in_picture_empty_text" msgid="5592688180792300051">"O‘rnatilgan ilovalar Tasvir ustida tasvir rejimi bilan mos emas"</string>
+    <string name="picture_in_picture_empty_text" msgid="5592688180792300051">"Oʻrnatilgan ilovalar Tasvir ustida tasvir rejimi bilan mos emas"</string>
     <string name="picture_in_picture_app_detail_summary" msgid="3478182415601436326">"Ilovalar ochiqligida yoki uni qoldirib ketganingizda (masalan, video tomosha qilishni davom ettirish uchun) bu ilova Tasvir ustida tasvir rejimida ishlashiga ruxsat bering. Bu oyna boshqa ishlatilayotgan ilovalar ustida ochiladi."</string>
     <string name="special_access" msgid="3513811566600299998">"Maxsus ruxsatlar"</string>
     <string name="string_concat" msgid="5897947064488902904">"<xliff:g id="PART1">%1s</xliff:g>, <xliff:g id="PART2">%2s</xliff:g>"</string>
-- 
2.17.1

